## Emails

- #â¬œ/ðŸ§¨| [[2021-03-30 Scratch]] | Call the NSLSC at 1 888 815-4514
![[Pasted image 20210324100902.png]]
- #â¬œ/âœ¨ | [[2021-03-31]] | Scratch cleanup infoserv (5155764604 keaton)

[[@Emil Karpsinki|Emil]] passed along an important [[training resource]] [[shotts2019LinuxCommandLine|Shotts 2019 The Linux Command Line]]. This will probably be useful as a [[training resource]] now that the CodeAcademy command-line fundamentals course is behind a paywall.

[[@Hendrik Poinar|Hendrik]] pointed out that my proposed [[BEAST2 Phylogeography Experiment]] will probably overlap with the work that [[@Sebastian Duchene]] is doing. We'll try to meet with him next week.
- #â¬œ/âœ¨ | [[2021-03-30 Scratch]] | Mock up [[BEAST2 Phylogeography Experiment]]
- #â¬œ/âœ¨ | [[2021-04-01]] | Meet with Sebastian Duchene.


### [[plague-phylogeography]] updates

Got some updates from [[@Leo Featherstone]] and [[@Sebastian Duchene]] about the [[bayesian dating analysis]] for [[plague-phylogeography]]. 

#### Dating

Two tips are not able to calibrated correctly:
- [[SAMN00715800]] | Strain 8291 | [[Black Death]]: [[East Smithfield]] | 1348-1350
- [[SAMEA3713715]] | Strain OBS137 | [[Plague of Marseille]] | 1720-1722

SAMEA3713715 wants to be younger, and SAMN00715800 wants to be older. In the worst case scenario, they'll be given a fixed date.

They are currently working on a [[Clock Model]]. Afterwards, they will:

> Try an approach where the recent and older samples have different rates, to attempt to accommodate time-dependency

Which I'm guess is a [[date randomization]] procedure? Actually on further reflection, I'm thinking this a variant of a [[Clock Model]], in which the [[Substitution Rate]] parameter will be modeled as having 2 states.

They have done a significant amount of work on the dating and have plenty of caveats. It will be framed as a [[model and prior sensitivity analysis]]. Which means that the results have to be interpreted in the context of the [[model assumptions]] and [[parameter uncertainy]]. ie. Different models and parameters are likely to produce different results.


#### Phylogeography

There is the problem of [[Sampling Bias|sampling bias]] in that some locations have lots of samples while others are very sparsely sampled. One approach might be to [[downsample]] each geographic location by its [[phylogenetic diversity]], potentially using [[Treemer]]. However, there are concerns about whether the correct geographic ranges have been specified. Sebastian says that I have grouped countries together (?) which they don't have in the metadata. He personally likes the [[Map|maps]], but would be well-complemented with [[Migration Plot|migration plots]] and [[Markov Reward]].

[[@Sebastian Duchene|Sebastian]] was interested in the [[isolation by distance]] plots.

> Try a model where geographic distance is treated as a predictor of migration rates in the model. If anything to say that simple linear distance is not a good predictor and that the story is more complicated than that.

So comparing two nodes and calculating the [[geographic distance]] between them. And that distance will be used as a [[predictor]] of the [[migration rate]] at that point. In the 'best' case, there is a correlation and it is a strong predictor. In the 'worst' case, it shows the complexity of the [[geographic history]] of plague. And perhaps also the complex [[ecology]], since so many hosts are involved that have different [[migration potential]]. In addition to the [[endemic-epidemic cycle]].

#### Follow-Up

1. **Problematic Samples ([[Black Death]] and [[Plague of Marseille]])**
	 - These samples have close relatives that were cut out in the [[Treemer]] sampling. For example, there are 3 other samples dated to 1720-1722 that form an epidemic clade with [[SAMN00715800]]. Since most of the temporal signal comes from the ancient samples, would it help if I made a new reduced subsample that contains all the [[Ancient]] genomes.

2. **[[Clock Model]]**
	> Try an approach where the recent and older samples have different rates, to attempt to accommodate time-dependency

	If I understand correctly, this would be a [[Clock Model]], in which the [[Substitution Rate]] parameter is modeled with 2 states (1 for 'old' samples, 1 for recent')?

3. **[[Phylogeography]]**
	Could you clarify what you mean by:
	>We wanted to confirm that we have the right geographic ranges that you are using, Kat. For example, you grouped a few countries together, which I donâ€™t think we have in the metadata.

	For ancestral state reconstruction on geographic location, I am using the variable "Province" from the metadata file as a discrete characteristic. For visualization, I use the corresponding columns "ProvinceLon" and "ProvinceLat". Does that help at all?

4. [[Markov Reward]]
	Thank you for sending along your paper on this! I read it briefly after our first meeting and will go through it again. 


## Meta

I would like a way to organize my inputs (#ðŸ“¥) in parallel with my [[Zotero]] collections. Ideally this would accomplish a few things:

- Mark the type of input (Article, Book, Video, etc.) | [[README#Inputs]]
	![](README.md#Tag%5CTaxonomy#Inputs)

- Mark the processing status (Inbox, Reading, Read, Consult)

	![](README.md#Tag%5CTaxonomy#Status)	

I learned more about embedding [[Markdown#Subheading|Subheadings]] in [[Markdown]] files.

- I did a little bit of [[Mermaid]] practice making a [[Flowchart]] of my [[Mermaid#Daily Obsidian Workflow|Daily Obsidian Workflow]].

## Life

My meetings have way too much natural light, need to wash and hang the curtains!
- #â¬œ/âœ¨ | [[2021-03-29]] | Curtains!

##  [[Plague Denmark]]

I'm working on the [[Chromosome Resequencing Experiment]] experiment for [[Plague Denmark]]. I'm searching for an efficient way to embed the results directly into the [[Markdown]] notes. I would like to be able to:

- Plot a bar graph
- Plot a line graph

I tested out the [[Mermaid#Gant Diagram|Gantt]] diagrams in Mermaid, and they look OK but are terribly inefficient to create. Instead, I would like to create make something like a [[Plotly]] chart and embed it in the [[Markdown]] as an [[iframe]].

- #â¬œ/âœ¨ Embed interactive graphs into [[Markdown]] files. 

I've had great success with this! Check out the [[Plotly]] note for more details.

Email update for [[@Ravneet Sidhu|Ravneet]]:

>Hi Ravneet,
>
> I'm experiencing Issues with [[Infoserv]] running [[MultiQC]] VERY slowly, so it's going run overnight. In the meantime I'm making some stats tables manually. Parsing the preseq output for library complexity is tedious so that will be done tomorrow! 
> 
>The objective for my 'experiment' today was:
>  1. Identify [[Yersinia pestis]]-positive samples for [[Resequencing]] based on:
>      - [[Genome coverage]], specifically the [[Chromosome]].
>      - Informative [[SNP|SNPs]].
>      - [[Library complexity]]
>	 
>Conclusions from today:
>  1. Our threshold of >=50% at a depth of 3X correlates nicely with capturing a sufficient number of informative SNPs. Using the Black Death Strain 8291 as a baseline, our target is to capture around 100 or more SNPs. There was only one sample where the coverage and SNPs didn't match (D62).
>  
>      ![[Pasted image 20210324164421.png]]
>      Interactive Version: <https://chart-studio.plotly.com/~ktmeaton/1>  
>      
>  2. The exhaustion/duplication estimates are looking promising for deeper sequencing. To be continued tomorrow.
>  
>Cheers,
>Kat

Tomorrow I'll need to continue this experiment.

- #â¬œ/âœ¨ | [[2021-03-25]] | Check [[MultiQC]] output of [[Chromosome Resequencing Experiment]].
- #â¬œ/âœ¨ | [[2021-03-25]] | Fill [[Preseq]] exhaustion estimates and finish [[Plotly]] chart.

## Updating Tags to Links

I'd like to try and switch over as many tags to links as possible in an automated fashion.

```bash
for file in `ls phd/**/*.md`;
do
  echo "file = $file"
done
```

  grep -P " # [A-Z,a-z,0-9].[A-Z,a-z,0-9,\-]*" '$file;
  
```bash
SAVEIFS=$IFS
IFS=$(echo \-en "\\n\\b")

for file in `ls phd/**/*.md`;
do
  grep -o -w -P " #[A-Z,a-z,0-9].[A-Z,a-z,0-9,\-]*" $file; 
done;

IFS=$SAVEIFS
  ```
  
  I'll have to remember this helpful piece of code for [[looping over file names with spaces in them]].
  
  ---

tags:
prev: [[2021-03-23 Scratch]]
next: [[2021-03-25 Scratch]]