## Emails

- #â¬œ/ðŸ“‹ | #2021-03-30 | Call the NSLSC at 1 888 815-4514
![[Pasted image 20210324100902.png]]
- #â¬œ/ðŸ“‹ | #2021-03-31 | Scratch cleanup infoserv (5155764604 keaton)
- #â¬œ/ðŸ“‹ | #2021-03-31 | Scratch cleanup Compute Canada

[[@Emil Karpsinki|Emil]] passed along an important [[training resource]] [[shotts2019LinuxCommandLine|Shotts 2019 The Linux Command Line]]. This will probably be useful as a [[training resource]] now that the CodeAcademy command-line fundamentals course is behind a paywall.

[[@Hendrik Poinar|Hendrik]] pointed out that my proposed [[BEAST2 Phylogeography Experiment]] will probably overlap with the work that [[@Sebastian Duchene]] is doing. We'll try to meet with him next week.
- #â¬œ/ðŸ“‹ | #2021-03-31 | Mock up [[BEAST2 Phylogeography Experiment]]
- #â¬œ/ðŸ“‹ | #2021-04-01 | Meet with Sebastian Duchene.


### [[plague-phylogeography]] updates

Got some updates from [[@Leo Featherstone]] and [[@Sebastian Duchene]] about the [[bayesian dating analysis]] for [[plague-phylogeography]]. 

#### Dating

Two tips are not able to calibrated correctly:
- SAMEA3713715 | Strain 8291 | [[Black Death]]: [[East Smithfield]] | 1348-1350
- SAMN00715800 | Strain OBS137 | [[Plague of Marseille]] | 1720-1722

SAMEA3713715 wants to be younger, and SAMN00715800 wants to be older. In the worst case scenario, they'll be given a fixed date.

They are currently working on a [[relaxed clock]]. Afterwards, they will:

> Try an approach where the recent and older samples have different rates, to attempt to accommodate time-dependency

Which I'm guess is a [[date randomization]] procedure? Actually on further reflection, I'm thinking this a variant of a [[relaxed clock]], in which the [[substiution rate]] parameter will be modeled as having 2 states.

They have done a significant amount of work on the dating and have plenty of caveats. It will be framed as a [[model and prior sensitivity analysis]]. Which means that the results have to be interpreted in the context of the [[model assumptions]] and [[parameter uncertainy]]. ie. Different models and parameters are likely to produce different results.


#### Phylogeography

There is the problem of [[Sampling Bias|sampling bias]] in that some locations have lots of samples while others are very sparsely sampled. One approach might be to [[downsample]] each geographic location by its [[phylogenetic diversity]], potentially using [[Treemer]]. However, there are concerns about whether the correct geographic ranges have been specified. Sebastian says that I have grouped countries together (?) which they don't have in the metadata. He personally likes the [[Map|maps]], but would be well-complemented with [[Migration Plots]] and [[Markov Reward]].

[[@Sebastian Duchene|Sebastian]] was interested in the [[isolation by distance]] plots.

> Try a model where geographic distance is treated as a predictor of migration rates in the model. If anything to say that simple linear distance is not a good predictor and that the story is more complicated than that.

So comparing two nodes and calculating the [[geographic distance]] between them. And that distance will be used as a [[predictor]] of the [[migration rate]] at that point. In the 'best' case, there is a correlation and it is a strong predictor. In the 'worst' case, it shows the complexity of the [[geographic history]] of plague. And perhaps also the complex [[ecology]], since so many hosts are involved that have different [[migration potential]]. In addition to the [[endemic-epidemic cycle]].

#### Follow-Up

1. **Problematic Samples ([[Black Death]] and [[Plague of Marseille]])**
	 - These samples have close relatives that were cut out in the [[Treemer]] sampling. For example, there are 3 other samples dated to 1720-1722 that form an epidemic clade with SAMN00715800. Since most of the temporal signal comes from the ancient samples, would it help if I made a new reduced subsample that contains all the [[Ancient]] genomes.

2. **[[Relaxed Clock]]**
	> Try an approach where the recent and older samples have different rates, to attempt to accommodate time-dependency

	If I understand correctly, this would be a [[relaxed clock]], in which the [[substiution rate]] parameter is modeled with 2 states (1 for 'old' samples, 1 for recent')?

3. **[[Phylogeography]]**
	Could you clarify what you mean by:
	>We wanted to confirm that we have the right geographic ranges that you are using, Kat. For example, you grouped a few countries together, which I donâ€™t think we have in the metadata.

	For ancestral state reconstruction on geographic location, I am using the variable "Province" from the metadata file as a discrete characteristic. For visualization, I use the corresponding columns "ProvinceLon" and "ProvinceLat". Does that help at all?

4. [[Markov Reward]]
	Thank you for sending along your paper on this! I read it briefly after our first meeting and will go through it again. 


## Meta

I would like a way to organize my inputs (#ðŸ“¥) in parallel with my [[Zotero]] collections. Ideally this would accomplish a few things:

- Mark the type of input (Article, Book, Video, etc.) | [[INDEX#Inputs]]
	![](INDEX#Tag\Taxonomy\#Inputs)

- Mark the processing status (Inbox, Reading, Read, Consult)

	![](INDEX#Tag\Taxonomy\#Status)	

I learned more about embedding [[Markdown#Subheading|Subheadings]] in [[Markdown]] files.

## Life

My meetings have way too much natural light, need to wash and hang the curtains!
- #â¬œ/ðŸ§¨ | #2021-03-24 | Curtains!
