# 2021-04-13 Scratch

## Emails

- #ðŸš‚ | NCBImeta Security Update 

## [[Plague Phylodynamics and Phylogeography]]

### [[Polytomy]]

I'm investigating extremely small branches across the tree. For example:

![[Short Branch Length 1.png]]

- 2.34e-8 and 2.345e-7 are very common small branches.

I'm curious what would happen if I use the ```--polytomy``` option in [[IQTREE]]? To my surprise, there are no [[Polytomy|polytomies]] in the output IQTREE, just extremely short branches.

- #âœ¨ | [[2021-04-13]] | Test polytomy option of IQTREE.

I will remove bootstrapping, and add the --polytomy option.

```bash
rm results/mugration/ results/clock/ results/parse_tree/ results/iqtree/ results/snippy_multi/

snakemake iqtree_all -np --profile profiles/infoserv --configfile results/config/snakemake.yaml
```

![[Short Branch Length 2.png]]

This collapsed internal branches into polytomies, but left ultra short tip branches at 2.34e-8. How do I know what a 0 length branch is? Presumably it's less than 1 site for the alignment length.
```bash
$cat ../plague-phylogeography/results/snippy_multi/all/snippy-core_chromosome.full.constant_sites.txt | \
awk -F "," '{print 1/($1 + $2 + $3 + $4)}'
 
2.34692e-07
```

Do I want to convert 0 length branches to true 0? Here are some interesting blog posts, [blog post 1](https://justinbagley.rbind.io/2016/03/01/dealing-negative-phylogenetic-branch-lengths-beast-starting-trees/) and [blog post 2](http://boopsboops.blogspot.com/2010/10/negative-branch-lengths-in-neighbour.html) to discuss this. These primarily deal with negative branch lengths but have some applications for tiny branches as well:

```bash
di2multi(phy, tol = 1e-08)
```

>which according to APE documentation â€œdeletes all branches smaller than â€˜tolâ€™ and collapses the corresponding dichotomies into a multichotomy.â€ This will take internal zero- or negative-length branches and collapse them. Here, phy is a class â€œphyloâ€ object containing your tree

Again, another mention of this:
[Zero-length branches are not treated as multichotomies, and thus may need to be collapsed (see di2multi)](https://rdrr.io/cran/ape/man/compute.brlen.html)

Also an idea from an R discussion:

>The presence of zero-lengthed terminal branches of results in an error since it makes little sense to have zero-rate branches.

Which is a good point, what is the purpose of having a terminal branch with length 0? Except I've noticed that it is possible from the output of IQTREE:

![[Short Branch Length 3.png]]

The newick data:
```text
((((((
  					GCA_015158755.1_ASM1515875v1_genomic:0.0000000000,
  					GCA_015159085.1_ASM1515908v1_genomic:0.0000000000):0.0000000000,
  				GCA_015190655.1_ASM1519065v1_genomic:0.0000000000):0.0000000000,
  			GCA_015159195.1_ASM1515919v1_genomic:0.0000000000):0.0000000000,
  		GCA_015158945.1_ASM1515894v1_genomic:0.0000000000):0.0000000000,
  	GCA_015159035.1_ASM1515903v1_genomic:0.0000000000):0.0000000000,
  GCA_015158875.1_ASM1515887v1_genomic:0.0000000000):0.0000000234,
```

Also good to note that [[IQTREE]] precision is ```1E-10```.

On a related note, I've discovered that [[Treemer]] has a flag to randomly resolve [[Polytomy|polytomies]].

### [[Singleton|Singleton Sites]]

I am going to also evaluate [[Singleton|singleton]] sites.

- #âœ¨ | [[2021-04-13]] | Evaluate singleton sites.

> [In the context of multiple sequence alignment and phylogeny reconstruction, singleton are sites at which only one of the species has a distinct nucleotide, whereas all others are the same. For parsimony methods, they are not informative, so sometimes are removed - though it doesn't change the estimated phylogeny, it just saves computation cost. This is not the case for maximum-likelihood (and probably also Bayesian) methods.](https://www.biostars.org/p/376899/)

[[Yang 2012 Molecular Phylogenetics Principles]]:
> Some sites are not useful for tree comparison by parsimony. For example, constant sites, for which the same nucleotide occurs in all species, have a character length of zero on any tree. Singleton sites, at which only one of the species has a distinct nucleotide, whereas all others are the same, can also be ignored, as the character length is always one.

### [[Missing Data]]

I think I need to edit the [[Jupyter Notebook]] that plots [[Missing Data]]. I think it would be more informative as a stacked/side bar chart with parsimony and singleton sites separate. I am quite pleased with this:

![[Missing Data Sites 2.png]]

## Other

- #ðŸ§¨ | [[2021-04-14]] | [[Ravneet Sidhu|Ravneet]] |  Prepare for meeting with Ravneet.
- #ðŸ§¨ | [[2021-04-16]] | [[Emil Karpinski|Emil]] |   Prepare for meeting with Emil.

## [[Obsidian]]

I added several new plugins to [[Obsidian]]:

### [[Emoji Toolbar]]
Hotkey Ctrl+J to input emojis ðŸ¦ˆ

### [[Dataview]]
Query and display note metadata.

#### Table

Literature:

```dataview
table Date, Authors
from "literature"
where Date > 2015
```

Tasks:

```dataview
table Project, Status, Due
from "tasks"
sort Due
```