---
# Bibliography
title: "A Palindromic RNA Sequence as a Common Breakpoint Contributor to Copy-Choice Recombination in SARS-COV-2"
authors: 
  - William R. Gallaher
date: "2020"
aliases: 
  - "Gallaher 2020"
  - "Gallaher (2020)"
  - "@gallaher2020PalindromicRNASequence"
bibtex: "gallaher2020PalindromicRNASequence"
doi: "10.1007/s00705-020-04750-z"
url: "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7394270/"

# Obsidian
type: Literature
tags: üì∞
---

# Gallaher (2020) A Palindromic RNA Sequence as a Common Breakpoint Contributor to Copy-Choice Recombination in SARS-COV-2

## Summary

[[William Gallaher]] investigated the genomic regions surrounding the [[Furin Cleavage site]] which is heavily suspected to have emerged via recombination. Immediately before this site is a tandem duplication: CAGACTCAGACT caused by polymerase stutter. This stutter can happen as the polymerase approaches a base-paired section of RNA, like a [[Stem Loop]] structure. As the helicase unwinds and "melts" the section ahead, the polymerase's process is impeded. 

Similar sequences were observed before inserted regions in ORF1a (<sub>3327</sub>CAGAC), M (<sub>26527</sub>CAGAT), and RBD (<sub>22839</sub>CAGAT).

In a single-stranded RNA virus, recombination does not typically occur via breakage and rejoining of strands. Instead, the more common mechanism is the polymerase slowing down during transcription and jumping to another template strand. This can take place in either direction of synthesis, although the majority of replications are of the positive strand from the negative template. 

In SARS-CoV--2, recombination regions/inserts, the breakpoint sequence is typically found at the 5' end. This indicates that the recombination occurred during positive strand synthesis from the negative strand template. The actual breakpoint sequence, is therefore on the template being read 3' to 5', implying the breakpoint sequence is 3'-GTCTG-5'. However, it's more convenient to define the sequence according to the positive strand.

- Regions that are highly suspected to have recombinant origins are typically preceded (5') by CAGAC/CAGAT.

---

> Automatically created by [`seldon`](https://github.com/ktmeaton/seldon).

- Fig. 3  RNA sequences of the common 5‚Äô terminus of the 3‚ÄôOH  region and a putative source of the furin insert RNA. A. RNA  sequence of the common 5‚Äô terminus of the 3‚ÄôOH region of SARSCoV-2, RaTG13, and BetaCoV/pan/P1L/2019.
- The RNA sequence  begins with the termination codon for the nucleocapsid (N) protein,  TAA, and continues through a region replete with five non-overlapping pentanucleotide palindromic sequences, indicated by brackets, including one each of the ‚Äúbreakpoint sequences‚Äù CAGAC and  CAGAT, highlighted in red type.
- Despite RNA  structure prediction programs, we have a poor understanding of these structures, but a correlate of the breakpoint  hypothesis would be that ‚Äúbreakpoint sequences‚Äù separate  structural regions of RNA as they do functional regions of  RNA and protein.
- Close examination of the RNA  sequence at the beginning of the 3‚ÄôOH region demonstrates  that SARS-CoV-2, RaTG13 and Pan_SL-CoV_GD/P1L  share a common identical sequence that is replete with palindromic sequences (Fig.¬†3a).
- Here, I present evidence that the RNA genome of SARSCoV-2 is organized into structural and functional blocks of  RNA information that are demarcated by short RNA breakpoint sequences that promote recombination at specific nonrandom locations within the viral genome.