---
# Bibliography
title: "Plasmid Profiler: Comparative Analysis of Plasmid Content in WGS Data"
authors: 
  - [[Adrian Zetner]]
  - [[Jennifer Cabral]]
  - [[Laura Mataseje]]
  - [[Natalie C. Knox]]
  - [[Philip Mabon]]
  - [[Michael Mulvey]]
  - [[Gary Van Domselaar]]
date: "2017"
aliases: 
  - "Zetner et al. 2017"
  - "Zetner et al. (2017)"
  - "@zetner2017PlasmidProfilerComparative"
bibtex: "zetner2017PlasmidProfilerComparative"
doi: "10.1101/121350"
url: "https://www.biorxiv.org/content/10.1101/121350v2"

# Obsidian
type: [[Literature]]
tags: ðŸ“°
---

# Zetner et al. (2017) Plasmid Profiler: Comparative Analysis of Plasmid Content in WGS Data

## Summary

> Automatically created by [`seldon`](https://github.com/ktmeaton/seldon).

- It is made available under  The copyright holder for this preprint (which was not this version posted March 28, 2017.
- Plasmid sequences identified with SRST2 are used to create a custom BLAST database  128  against which 116 plasmid replicon sequences (Center for Genomic Epidemiologyâ€™s  129  PlasmidFinder database available at: https://cge.cbs.dtu.dk//services/data.php) are queried  130  using MegaBLAST (Morgulis,A., Coulouris,G., et al. 2008).
- The user-modifiable replicon and genes of interest database is a  101  multi-fasta file with plasmid replicon sequences from PlasmidFinder (Center for Genomic  102  Epidemiologyâ€™s PlasmidFinder database available at: https://cge.cbs.dtu.dk//services/data.php)  103  along with user-defined nucleotide sequences for genes of interest.
- Conclusion  172  Plasmid Profiler is a useful comparative tool for visualization of plasmid content in WGS data  173  from bacterial isolates without resorting to difficult and uncertain assembly methods or the use  174  of long-read sequencing technologies.
- The pipeline is  20  designed to rapidly identify plasmid sequences by mapping reads to a plasmid reference  21  sequence database.