<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Eaton, K., Featherstone, L., Duchene, S., Carmichael, A., Varlık, N., Golding, G. B., Holmes, E., Poinar, H. N.">
  <title>Plagued by a cryptic clock</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reset.css">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/theme/black.css" id="theme">
  <link rel="stylesheet" href="pandoc/templates/revealjs-obsidian/beap_2021-10-01.css"/>

</head>
<body>
  <div class="reveal">
    <div class="slides">


  <section id="title-slide">
  <h1 class="title">Plagued by a cryptic clock</h1>
  <p class="subtitle">Insight and issues from the global phylogeny of <em>Yersinia pestis</em></p>
  <p class="author"><small>Eaton, K., Featherstone, L., Duchene, S., Carmichael, A., Varlık, N., Golding, G. B., Holmes, E., Poinar, H. N.</small></p>
  <p>BEAP</p>

  <p class="date">2021 October 01</p>

        <img height=200px src=https://rawcdn.githack.com/ktmeaton/plague-phylogeography/e74d7b0d732023d109e1abbefde3305eae589ebd/docs/images/thumbnail_DHSI2020.png>

</section>



<section>
<section id="introduction" class="title-slide slide level2">
<h2>Introduction</h2>

</section>
<section id="concept-1.-disease-exposure" class="slide level3">
<h3>Concept 1. Disease Exposure</h3>
<hr>
<ul>
<li>How do we contract an infectious disease?</li>
<li>How do we spread it?</li>
<li>Why does this vary across time and space?</li>
</ul>
</section>
<section id="concept-2.-disease-experience" class="slide level3">
<h3>Concept 2. Disease Experience</h3>
<hr>
<ul>
<li>Are the symptoms mild?</li>
<li>Does it become endemic and part of our everyday life?</li>
<li>Is there catastrophic mortality?</li>
<li>Does it radically change society?</li>
</ul>
</section>
<section id="zoonoses" class="slide level3">
<h3>Zoonoses</h3>
<hr>
<ul>
<li>Diseases that spread from non-human animals to humans.</li>
<li>Enigmatic epidemiology that is often poorly understood.
<ul>
<li>COVID-19</li>
<li>Malaria</li>
<li>Influenza</li>
<li>Brucellosis</li>
<li>Lyme Disease</li>
<li>Plague</li>
</ul></li>
</ul>
</section>
<section id="plague" class="slide level3">
<h3>Plague</h3>
<hr>
<ol type="1">
<li><strong>Prolific presence in human populations, at least since the Bronze Age (~3000 YBP).</strong></li>
</ol>
<figure>
<img data-src="xu2019HistoricalGenomicData_fig1.jpg" id="fig:xu_2019" alt="Figure 1: Geographic distribution of human plague cases (1772 - 2014).[1]" /><figcaption aria-hidden="true">Figure 1: Geographic distribution of human plague cases (1772 - 2014).<span class="citation" data-cites="xu2019HistoricalGenomicData"><sup>[1]</sup></span></figcaption>
</figure>
</section>
<section id="plague-1" class="slide level3">
<h3>Plague</h3>
<hr>
<ol start="2" type="1">
<li><strong>Unnerving virulence and mortality.</strong>
<ul>
<li>The medieval Black Death (1346-1353) killed more than 50% of Europe’s population..<span class="citation" data-cites="benedictow2021CompleteHistoryBlack"><sup>[2]</sup></span></li>
<li>Untreated pneumonic plague has a case fatality rate of nearly 100%.<span class="citation" data-cites="prentice2007Plague salam2020DeathsAssociatedPneumonic"><sup>[3,4]</sup></span></li>
</ul></li>
<li><strong>It keeps coming back!</strong>
<ul>
<li>First Pandemic (6<sup>th</sup> - 8<sup>th</sup> Century CE)</li>
<li>Second Pandemic (14<sup>th</sup> - 19<sup>th</sup> Century CE)</li>
<li>Third Pandemic (19<sup>th</sup> - 20<sup>th</sup> Century CE)</li>
</ul></li>
<li><strong>A cryptic clock.</strong>
<ul>
<li>Debated whether molecular clock techniques can produce informative rates and dates.</li>
<li>An extremely slow evolutionary rate coupled with extreme rate variation.</li>
</ul></li>
</ol>
</section>
<section id="objectives" class="slide level3">
<h3>Objectives</h3>
<ol type="1">
<li>Is there any way to <strong>estimate robust rates and dates</strong> for <em>Y. pestis</em>?</li>
<li>Do I have <strong>anything new to say</strong> about the history and epidemiology of plague?</li>
</ol>
</section></section>
<section>
<section id="materials-and-methods" class="title-slide slide level2">
<h2>Materials and Methods</h2>

</section>
<section id="materials-and-methods-1" class="slide level3">
<h3>Materials and Methods</h3>
<hr>
<aside class="notes">
<ul>
<li>Bayesian (BEAST) analysis was inconclusive (strict or relaxed).<br></li>
<li>Poor sampling, lack of convergence, low ESS.<br></li>
<li>Why? Too much rate variation!<br></li>
<li>Reducing the number of samples, reducing tip date uncertainty, fixing the tree topology.<br></li>
<li>Rates and dates are not informative!<br></li>
<li>Pictures of Sebastian and Leo.<br></li>
</ul>
</aside>
<ol type="1">
<li><strong>601 publicly available <em>Y. pestis</em> genomes.</strong>
<ul>
<li>540 modern (89.9%): 20<sup>th</sup> to 21<sup>st</sup> century CE.</li>
<li>61 ancient (10.1%): 3<sup>rd</sup> millenium BCE to 18<sup>th</sup> century CE.</li>
</ul></li>
<li><strong>Multiple alignment of the core genome.</strong></li>
<li><strong>Estimated a maximum-likelihood phylogeny.</strong></li>
<li><strong>Bayesian Evaluation of Temporal Signal (BETS).</strong>
<ul>
<li>Strict Clock</li>
<li>Relaxed Clock</li>
<li>Global Clock</li>
<li>Population Clocks<span class="citation" data-cites="cui2013HistoricalVariationsMutation"><sup>[5]</sup></span></li>
</ul></li>
<li><strong>Estimated rates and dates for populations with temporal signal.</strong></li>
</ol>
</section>
<section id="population-structure" class="slide level3">
<h3>Population Structure</h3>
<hr>
<figure>
<img data-src="divtree.svg" id="fig:divtree" alt="Figure 2: The maximum-likelihood phylogeny of 601 Yersinia pestis genomes." /><figcaption aria-hidden="true">Figure 2: The maximum-likelihood phylogeny of 601 <em>Yersinia pestis</em> genomes.</figcaption>
</figure>
</section>
<section id="molecular-clocks" class="slide level3">
<h3>Molecular Clocks</h3>
<hr>
<ul>
<li>The global clock analysis (all samples) was VERY unstable.</li>
</ul>
<figure>
<img data-src="rtt.svg" id="fig:rtt" height="400" alt="Figure 3: Root-to-tip regression of 601 Yersinia pestis genomes using the maximum-likelihood phylogeny." /><figcaption aria-hidden="true">Figure 3: Root-to-tip regression of 601 <em>Yersinia pestis</em> genomes using the maximum-likelihood phylogeny.</figcaption>
</figure>
</section>
<section id="molecular-clocks-1" class="slide level3">
<h3>Molecular Clocks</h3>
<hr>
<ul>
<li>Tried reducing sources of rate variation:
<ul>
<li>Decreasing the number of samples.</li>
<li>Fixing tip dates.</li>
<li>Fixing tree topology.</li>
</ul></li>
</ul>
<blockquote>
<p>The global diversity of <em>Y. pestis</em> is poorly modeled by a single molecular clock.</p>
</blockquote>
<hr>
<ul>
<li>Population clocks <strong>recovered temporal signal</strong> in 9 out of 12 populations.</li>
<li>For all populations, the relaxed clock outerperformed the strict clock.</li>
</ul>
</section></section>
<section>
<section id="the-phylodynamic-threshold" class="title-slide slide level2">
<h2>The Phylodynamic Threshold</h2>
<aside class="notes">
<ul>
<li>SARS-CoV-2: Is a virus, thus fast-evolving, threshold reached in 2 months of the start of the pandemic.</li>
</ul>
</aside>
<blockquote>
<p><em>“The point in time at which sufficient molecular evolutionary change has accumulated in available genome samples to obtain robust phylodynamic estimates.”</em><br> <br> <em>“Before the phylodynamic threshold is reached, genomic variation is so low that even large amounts of genome sequences may be insufficient to estimate the virus’s evolutionary rate and the time scale of an outbreak.”</em><br> <br> - Duchene et al. (2020).<span class="citation" data-cites="duchene2020TemporalSignalPhylodynamic"><sup>[6]</sup></span></p>
</blockquote>
</section>
<section id="rates-of-evolutionary-change" class="slide level3" data-transition="none">
<h3 data-transition="none">Rates of Evolutionary Change</h3>
<hr>
<ul>
<li><em>Y. pestis</em> has one of the slowest substitution rates among bacterial pathogens.<span class="citation" data-cites="duchene2016GenomescaleRatesEvolutionary"><sup>[7]</sup></span></li>
<li><em>Y. pestis</em> has substantial rate variation.<span class="citation" data-cites="cui2013HistoricalVariationsMutation"><sup>[5]</sup></span></li>
</ul>
<figure>
<img data-src="rates_mcc_tree.svg" id="fig:rates_mcc_tree" height="400" alt="Figure 4: Substitution rates across the maximum-clade credibility (MCC) trees." /><figcaption aria-hidden="true">Figure 4: Substitution rates across the maximum-clade credibility (MCC) trees.</figcaption>
</figure>
</section>
<section id="rates-of-evolutionary-change-1" class="slide level3" data-transition="none">
<h3 data-transition="none">Rates of Evolutionary Change</h3>
<hr>
<ul>
<li><em>Y. pestis</em> has one of the slowest substitution rates among bacterial pathogens.<span class="citation" data-cites="duchene2016GenomescaleRatesEvolutionary"><sup>[7]</sup></span></li>
<li><em>Y. pestis</em> has substantial rate variation.<span class="citation" data-cites="cui2013HistoricalVariationsMutation"><sup>[5]</sup></span></li>
</ul>
<figure>
<img data-src="rates_mcc_tree_no_outliers.svg" id="fig:rates_mcc_tree_no_outliers" height="400" alt="Figure 5: Substitution rates across the maximum-clade credibility (MCC) trees with outliers removed." /><figcaption aria-hidden="true">Figure 5: Substitution rates across the maximum-clade credibility (MCC) trees with outliers removed.</figcaption>
</figure>
</section>
<section id="why-do-we-care" class="slide level3">
<h3>Why Do We Care?</h3>
<ul>
<li>Rates and dates of short-term events will be meaningless.
<ul>
<li>Ex. Epidemics, recently emerged populations. <br></li>
</ul></li>
</ul>
<blockquote>
<p>But what does “short-term” mean?</p>
</blockquote>
</section>
<section id="no-temporal-signal" class="slide level3">
<h3>No Temporal Signal</h3>
<hr>
<aside class="notes">
<ul>
<li>3 populations had insufficient phylogenetic and temporal information.<br></li>
</ul>
</aside>
<ul>
<li>Three populations have no temporal signal (Figure  6).</li>
<li>Shortest sampling time frames.</li>
<li>Node density is highest close to the root.</li>
</ul>
<blockquote>
<p>These populations have rates and dates that are <strong>non-informative</strong>.<br> The <strong>phylodynamic threshold</strong> of <em>Y. pestis</em> is greater than 50 years. <br> Comparisons over shorter time scales will have <strong>limited resolving power</strong>.</p>
</blockquote>
<figure>
<img data-src="no_temporal_signal.svg" id="fig:no_temporal_signal" alt="Figure 6: Maximum-likelihood phylogenies of Yersinia pestis populations with no detectable temporal signal. Grey branches indicate outliers." /><figcaption aria-hidden="true">Figure 6: Maximum-likelihood phylogenies of <em>Yersinia pestis</em> populations with no detectable temporal signal. Grey branches indicate outliers.</figcaption>
</figure>
</section>
<section id="epidemic-analysis" class="slide level3">
<h3>Epidemic Analysis</h3>
<hr>
<aside class="notes">
<blockquote>
<p><strong>Solution:</strong> Other genetic markers?</p>
</blockquote>
</aside>
<blockquote>
<p>Intra-epidemic diversity is poorly resolved by a core genome SNP phylogeny</p>
</blockquote>
<ul>
<li>Black Death (1346 - 1353) <em>Y. pestis</em> isolates are indistinguishable clones.<span class="citation" data-cites="spyrou2016HistoricalPestisGenomes"><sup>[8]</sup></span></li>
</ul>
<figure>
<img data-src="Map%20Black%20Death.png" id="fig:map_black_death" height="300" alt="Figure 7: Spread map of the Black Death.[9]" /><figcaption aria-hidden="true">Figure 7: Spread map of the Black Death.<span class="citation" data-cites="benedictow2004BlackDeath13461353"><sup>[9]</sup></span></figcaption>
</figure>
</section>
<section id="the-third-pandemic" class="slide level3">
<h3>The Third Pandemic</h3>
<hr>
<aside class="notes">
<ul>
<li><strong>Solution:</strong> Increased sampling of long internal branches.</li>
</ul>
</aside>
<ul>
<li>The events leading up to the Third Pandemic (1.ORI) are highly uncertain.<br></li>
<li>When and how, did the epidemiology of plague change from localized epidemics in Yunnan Province, China to a global pandemic?</li>
</ul>
<figure>
<img data-src="divtree_1.IN_1.ORI.svg" id="fig:divtree_1.IN_1.ORI" height="350" alt="Figure 8: Third Pandemic Origins" /><figcaption aria-hidden="true">Figure 8: Third Pandemic Origins</figcaption>
</figure>
</section></section>
<section>
<section id="ancient-dna" class="title-slide slide level2">
<h2>Ancient DNA</h2>

</section>
<section id="temporal-signal-with-ancient-dna" class="slide level3">
<h3>Temporal Signal With Ancient DNA</h3>
<aside class="notes">
<ul>
<li>Second Pandemic (1.PRE) starts with a polytomy, root position is incorrectly inferred.</li>
</ul>
</aside>
<hr>
<p>All (3) ancient <em>Y. pestis</em> populations had temporal signal:</p>
<ul>
<li>1.PRE | Second Pandemic (14<sup>th</sup> - 19 <sup>th</sup> century CE)</li>
<li>0.ANT4 | First Pandemic (6<sup>th</sup> - 8 <sup>th</sup> century CE)</li>
<li>0.PRE | Bronze Age (3<sup>rd</sup> - 1<sup>st</sup> Millennia BCE)</li>
</ul>
<figure>
<img data-src="temporal_signal_ancient.svg" id="fig:temporal_signal_ancient" alt="Figure 9: Maximum-likelihood phylogenies of ancient Y. pestis populations with detectable temporal signal. Grey branches indicate outliers." /><figcaption aria-hidden="true">Figure 9: Maximum-likelihood phylogenies of ancient <em>Y. pestis</em> populations with detectable temporal signal. Grey branches indicate outliers.</figcaption>
</figure>
</section>
<section id="what-can-we-learn" class="slide level3">
<h3>What Can We Learn?</h3>
<aside class="notes">
<ul>
<li>Example: What if we tried to reconstruct COVID pandemic just from European data? 2020, Italy.</li>
</ul>
</aside>
<hr>
<ul>
<li>Informative rates and dates can be obtained when <strong>the sampling time frame is &gt;500 years</strong>.</li>
<li>Node age <strong>uncertainty can be 100+ years</strong> (95% HPD).</li>
<li>Genetic evidence suggests that the <strong>plague pandemics may have earlier origins</strong>.</li>
<li>Most evidence of historical plague comes from European sources.</li>
<li>New scholarship of non-European sources is rewriting these narratives.<span class="citation" data-cites="varlik2020PlagueThatNever"><sup>[10]</sup></span></li>
</ul>
<p><br></p>
<div id="tbl:node_dating_ancient">
<table>
<caption>Table 1: Node dating of key historical plague events.</caption>
<thead>
<tr class="header">
<th style="text-align: center;">Population</th>
<th style="text-align: center;">Event</th>
<th style="text-align: center;">tMRCA (Lower)</th>
<th style="text-align: center;">tMRCA (Upper)</th>
<th style="text-align: center;">Historical Start</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">1.PRE</td>
<td style="text-align: center;">Second Pandemic</td>
<td style="text-align: center;">1214</td>
<td style="text-align: center;">1315</td>
<td style="text-align: center;">1346</td>
</tr>
<tr class="even">
<td style="text-align: center;">0.ANT4</td>
<td style="text-align: center;">First Pandemic</td>
<td style="text-align: center;">272</td>
<td style="text-align: center;">466</td>
<td style="text-align: center;">541</td>
</tr>
<tr class="odd">
<td style="text-align: center;">0.PRE</td>
<td style="text-align: center;">Bronze Age</td>
<td style="text-align: center;">-3098</td>
<td style="text-align: center;">-2786</td>
<td style="text-align: center;">–</td>
</tr>
</tbody>
</table>
</div>
</section></section>
<section>
<section id="modern-dna" class="title-slide slide level2">
<h2>Modern DNA</h2>

</section>
<section id="temporal-signal-with-modern-dna" class="slide level3">
<h3>Temporal Signal With Modern DNA</h3>
<aside class="notes">
<ul>
<li>Uncertainty always increases further back in time, but for 2.MED uncertainty gets really big really fast.</li>
<li>Insert time trees here?</li>
<li>High uncertainty in the root position, leading to large confidence intervals.</li>
<li>Use the branching order in the maximum likelihood phylogeny to help root it?</li>
<li><span class="citation" data-cites="morelli2010YersiniaPestisGenome"><sup>[11]</sup></span><br><span class="citation" data-cites="cui2013HistoricalVariationsMutation"><sup>[5]</sup></span><br>This Study</li>
<li>3rd pandemic date originally 1894, pushed back to 1855. Some argue as early as 1722.</li>
</ul>
</aside>
<hr>
<ul>
<li>Two modern populations had temporal signal, despite having no ancient DNA calibrations.
<ul>
<li>The Third Pandemic (1.ORI), global.</li>
<li><em>Medievalis</em> (2.MED), Asia.</li>
</ul></li>
<li>Have high epidemiological significance:
<ul>
<li>High mortality epidemics.</li>
<li>Extensive and rapid geographic spread.</li>
</ul></li>
</ul>
<figure>
<img data-src="temporal_signal_modern.svg" id="fig:temporal_signal_modern" alt="Figure 10: Maximum-likelihood phylogenies of modern Y. pestis populations with detectable temporal signal. Grey branches indicate outliers." /><figcaption aria-hidden="true">Figure 10: Maximum-likelihood phylogenies of modern <em>Y. pestis</em> populations with detectable temporal signal. Grey branches indicate outliers.</figcaption>
</figure>
</section>
<section id="temporal-signal-with-modern-dna-1" class="slide level3">
<h3>Temporal Signal With Modern DNA</h3>
<hr>
<ul>
<li>As more data becomes available, the <strong>node dates are becoming younger</strong>.<span class="citation" data-cites="pisarenko2021YersiniaPestisStrains"><sup>[12]</sup></span></li>
</ul>
<p><br></p>
<div id="tbl:node_dating_modern">
<table>
<caption>Table 2: Node dating of modern <em>Yersinia pestis</em> populations.</caption>
<colgroup>
<col style="width: 11%" />
<col style="width: 18%" />
<col style="width: 5%" />
<col style="width: 16%" />
<col style="width: 23%" />
<col style="width: 23%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Population</th>
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Study</th>
<th style="text-align: center;">Samples</th>
<th style="text-align: center;">tMRCA (Lower)</th>
<th style="text-align: center;">tMRCA (Upper)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><br>1.ORI<br></td>
<td style="text-align: center;"><br>Third Pandemic<br></td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">6<br>17<br>93<br>117</td>
<td style="text-align: center;">-381<br>1735<br>1742<br>1806</td>
<td style="text-align: center;">1738<br>1863<br>1842<br>1901</td>
</tr>
<tr class="even">
<td style="text-align: center;"><br>2.MED<br></td>
<td style="text-align: center;"><br><em>Medievalis</em><br></td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">2<br>25<br>68<br>116</td>
<td style="text-align: center;">-638<br>1450<br>1298<br>1560</td>
<td style="text-align: center;">1715<br>1750<br>1582<br>1845</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="timeline-of-the-third-pandemic" class="slide level3">
<h3>Timeline of The Third Pandemic</h3>
<aside class="notes">
<img data-src="divtree_1.IN_1.ORI.svg" title="fig:" height="350" alt="Third Pandemic Origins" />
</aside>
<hr>
<ul>
<li>We estimate that the Third Pandemic (1.ORI) <strong>emerged between 1806 and 1901 (1885)</strong>.</li>
<li>This <strong>aligns with historical evidence</strong> from Chinese epidemiologists<span class="citation" data-cites="xu2014WetClimateTransportation"><sup>[13]</sup></span> <span class="citation" data-cites="tan2002AtlasPlagueIts"><sup>[14]</sup></span>.</li>
<li>1772 - 1800: Human plague cases are isolated to Yunnan, China.</li>
<li>1880 - 1900: Plague spreads to the south-eastern provinces.</li>
<li>1894: Plague spreads worldwide out of Hong Kong.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td style="text-align: center;"><img data-src="divtree_1.IN_1.ORI.svg" width="500" alt="Third Pandemic Origins" /></td>
<td style="text-align: center;"><img data-src="third_pandemic_origins.png" width="300" alt="Third Pandemic Origins" /></td>
</tr>
</tbody>
</table>
</section>
<section id="medievalis" class="slide level3">
<h3><em>Medievalis</em></h3>
<hr>
<ul>
<li>We estimate that <em>Medievalis</em> (2.MED) <strong>emerged between 1560 and 1845 (1796)</strong>.</li>
<li>This <strong>aligns with historical evidence</strong>.<span class="citation" data-cites="kahan1979SocialAspectsPlague hashemishahraki2016PlagueIranIts eroshenko2019CirculationYersiniaPestis"><sup>[15–17]</sup></span></li>
<li>Late 1798: Human plague cases are observed in the Caucasus.</li>
<li>1800s: Plague cases throughout the Caspian Sea region.</li>
<li>1900s-2000s: Widespread dispersal across Asia.</li>
</ul>
<figure>
<img data-src="map_2.MED.svg" alt="The geographic distribution of Medievalis (2.MED)." /><figcaption aria-hidden="true">The geographic distribution of <em>Medievalis</em> (2.MED).</figcaption>
</figure>
</section>
<section id="whodunnit" class="slide level3">
<h3>Whodunnit?</h3>
<hr>
<ul>
<li><em>Medievalis</em> (2.MED), <em>Pestoides</em> (0.PE), or the Second Pandemic (1.PRE)?</li>
</ul>
<p><img data-src="Map%200.PE%201.PRE%202.MED.png" height="400" /></p>
</section></section>
<section>
<section id="good-temporal-signal-good-node-dates" class="title-slide slide level2">
<h2>Good Temporal Signal<br>≠<br>Good Node Dates</h2>

</section>
<section id="node-date-conflicts" class="slide level3">
<h3>Node Date Conflicts</h3>
<hr>
<ul>
<li>Several populations have conflicting dates for their tMRCA.</li>
<li>These populations are not monophyletic.</li>
<li>Ex. <em>Antiqua</em> (0.ANT) is ancestral to the First Pandemic (0.ANT4).</li>
<li>The tMRCA of <em>Antiqua</em> (0.ANT) incorrectly post-dates the First Pandemic (0.ANT4).</li>
</ul>
<figure>
<img data-src="date_conflict.svg" id="fig:date_conflict" height="400" alt="Figure 11: Node dating conflicts between the ancestral Antiqua (0.ANT) and the First Pandemic (0.ANT4)." /><figcaption aria-hidden="true">Figure 11: Node dating conflicts between the ancestral <em>Antiqua</em> (0.ANT) and the First Pandemic (0.ANT4).</figcaption>
</figure>
</section>
<section id="node-date-conflicts-1" class="slide level3">
<h3>Node Date Conflicts</h3>
<hr>
<ul>
<li>Node density is low close to the root.</li>
<li>Node dates are underestimated (too young). Can be off by a 1000 years!</li>
</ul>
<blockquote>
<p>Insufficient sample sizes (1.ANT).<br> Insufficient geographic sampling (0.ANT).<br> Extensive rate variation (0.PE) with no ancient DNA calibrations.</p>
</blockquote>
<figure>
<img data-src="suspicious_dates.svg" id="fig:suspicious_dates" alt="Figure 12: Maximum-likelihood phylogenies of modern Y. pestis populations lacking internal calibrations. Grey branches indicate outliers." /><figcaption aria-hidden="true">Figure 12: Maximum-likelihood phylogenies of modern <em>Y. pestis</em> populations lacking internal calibrations. Grey branches indicate outliers.</figcaption>
</figure>
</section>
<section id="arrival-of-plague-in-africa" class="slide level3">
<h3>Arrival of Plague in Africa</h3>
<aside class="notes">
<ul>
<li>Plague in Africa is really important for public health research = high interest.</li>
<li>But the genetic evidence is very flimsy for estimating a continental tMRCA.</li>
</ul>
</aside>
<hr>
<ul>
<li>Africa accounts for more than 90% of all human plague cases<span class="citation" data-cites="munyenyiwa2019PlagueZimbabwe1974"><sup>[18]</sup></span> .</li>
<li>But only 1.5% of <em>Y. pestis</em> genomes are from Africa.</li>
<li>Heavily debated when plague first arrived in Central and East Africa (1.ANT).</li>
</ul>
<p><br></p>
<table>
<thead>
<tr class="header">
<th style="text-align: center;">Population</th>
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Study</th>
<th style="text-align: center;">Samples</th>
<th style="text-align: center;">tMRCA (Lower)</th>
<th style="text-align: center;">tMRCA (Upper)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;"><br>1.ANT<br></td>
<td style="text-align: center;"><br>African <em>Antiqua</em><br></td>
<td style="text-align: center;">TBD</td>
<td style="text-align: center;">2<br>2<br>4<br></td>
<td style="text-align: center;">-4964<br>1377<br>1655</td>
<td style="text-align: center;">1322<br>1650<br>1835</td>
</tr>
</tbody>
</table>
<p><br></p>
<blockquote>
<p>Genetic dates for the tMRCA of 1.ANT in Africa are non-informative!<br> They should not be used as a basis for historical or archaeological interpretation.<span class="citation" data-cites="green2018PuttingAfricaBlack"><sup>[19]</sup></span></p>
</blockquote>
</section></section>
<section>
<section id="conclusions" class="title-slide slide level2">
<h2>Conclusions</h2>

</section>
<section id="methodology" class="slide level3">
<h3>Methodology</h3>
<hr>
<blockquote>
<p><strong>Poor performance of a global molecular clock.</strong><br> - Intensive global sampling → More genetic diversity → More rate variation.<br> - Partitioning the data by population stabilizes clock estimates.</p>
</blockquote>
<blockquote>
<p><strong>The phylodynamic threshold is very important!</strong><br> - A short sampling time frame (&lt;50 years) led to non-informative rates and dates.<br> - Longer sampling time frames (&gt;90 years) led to detectable temporal signal.</p>
</blockquote>
<blockquote>
<p><strong>Temporal signal does not mean <em>all</em> dates will be informative.</strong><br> - Insufficient sampling produces conflicting node dates.<br> - Can be detected in trees with few nodes close to the root.</p>
</blockquote>
</section>
<section id="plague-history" class="slide level3">
<h3>Plague History</h3>
<blockquote>
<p>The tMRCAs of <strong>ancient plague pandemics are older</strong> than previously thought.<br> - Challenges Eurocentric views of plague history.</p>
</blockquote>
<blockquote>
<p>The tMRCAs of <strong>modern plague are younger</strong> than previously thought.<br> - Aligns with historical evidence.</p>
</blockquote>
</section></section>
<section id="references" class="title-slide slide level2">
<h2>References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-xu2019HistoricalGenomicData" class="csl-entry" role="doc-biblioentry">
1. Xu, L., Stige, L. C., Leirs, H., Neerinckx, S., Gage, K. L., Yang, R., Liu, Q., Bramanti, B., Dean, K. R., Tang, H., Sun, Z., Stenseth, N. C., &amp; Zhang, Z. (2019). Historical and genomic data reveal the influencing factors on global transmission velocity of plague during the <span>Third Pandemic</span>. <em>Proceedings of the National Academy of Sciences</em>, <em>116</em>(24), 11833–11838. <a href="https://doi.org/10.1073/pnas.1901366116">https://doi.org/10.1073/pnas.1901366116</a>
</div>
<div id="ref-benedictow2021CompleteHistoryBlack" class="csl-entry" role="doc-biblioentry">
2. Benedictow, O. J. (2021). <em>The <span>Complete History</span> of the <span>Black Death</span></em>. <span>Boydell Press</span>.
</div>
<div id="ref-prentice2007Plague" class="csl-entry" role="doc-biblioentry">
3. Prentice, M. B., &amp; Rahalison, L. (2007). Plague. <em>The Lancet</em>, <em>369</em>(9568), 1196–1207. <a href="https://doi.org/10.1016/S0140-6736(07)60566-2">https://doi.org/10.1016/S0140-6736(07)60566-2</a>
</div>
<div id="ref-salam2020DeathsAssociatedPneumonic" class="csl-entry" role="doc-biblioentry">
4. Salam, A. P., Rojek, A., Cai, E., Raberahona, M., &amp; Horby, P. (2020). Deaths <span>Associated</span> with <span>Pneumonic Plague</span>, 1946 - <span>Volume</span> 26, <span>Number</span> 10 2020 - <span>Emerging Infectious Diseases</span> journal - <span>CDC</span>. <em>Emerging Infectious Diseases</em>, <em>26</em>(10), 2432–2434. <a href="https://doi.org/10.3201/eid2610.191270">https://doi.org/10.3201/eid2610.191270</a>
</div>
<div id="ref-cui2013HistoricalVariationsMutation" class="csl-entry" role="doc-biblioentry">
5. Cui, Y., Yu, C., Yan, Y., Li, D., Li, Y., Jombart, T., Weinert, L. A., Wang, Z., Guo, Z., Xu, L., Zhang, Y., Zheng, H., Qin, N., Xiao, X., Wu, M., Wang, X., Zhou, D., Qi, Z., Du, Z., … Yang, R. (2013). Historical variations in mutation rate in an epidemic pathogen, <span><em>Yersinia</em></span><span> <em>Pestis</em></span>. <em>Proceedings of the National Academy of Sciences</em>, <em>110</em>(2), 577–582. <a href="https://doi.org/10.1073/pnas.1205750110">https://doi.org/10.1073/pnas.1205750110</a>
</div>
<div id="ref-duchene2020TemporalSignalPhylodynamic" class="csl-entry" role="doc-biblioentry">
6. Duchene, S., Featherstone, L., Haritopoulou-Sinanidou, M., Rambaut, A., Lemey, P., &amp; Baele, G. (2020). Temporal signal and the phylodynamic threshold of <span>SARS</span>-<span>CoV</span>-2. <em>Virus Evolution</em>, <em>6</em>(2). <a href="https://doi.org/10.1093/ve/veaa061">https://doi.org/10.1093/ve/veaa061</a>
</div>
<div id="ref-duchene2016GenomescaleRatesEvolutionary" class="csl-entry" role="doc-biblioentry">
7. Duchêne, S., Holt, K. E., Weill, F.-X., Le Hello, S., Hawkey, J., Edwards, D. J., Fourment, M., &amp; Holmes, E. C. (2016). Genome-scale rates of evolutionary change in bacteria. <em>Microbial Genomics</em>, <em>2</em>(11). <a href="https://doi.org/10.1099/mgen.0.000094">https://doi.org/10.1099/mgen.0.000094</a>
</div>
<div id="ref-spyrou2016HistoricalPestisGenomes" class="csl-entry" role="doc-biblioentry">
8. Spyrou, M. A., Tukhbatova, R. I., Feldman, M., Drath, J., Kacki, S., Beltrán de Heredia, J., Arnold, S., Sitdikov, A. G., Castex, D., Wahl, J., Gazimzyanov, I. R., Nurgaliev, D. K., Herbig, A., Bos, K. I., &amp; Krause, J. (2016). Historical <span><em>Y</em></span><span><em>. Pestis</em></span> <span>Genomes Reveal</span> the <span>European Black Death</span> as the <span>Source</span> of <span>Ancient</span> and <span>Modern Plague Pandemics</span>. <em>Cell Host &amp; Microbe</em>, <em>19</em>(6), 874–881. <a href="https://doi.org/10.1016/j.chom.2016.05.012">https://doi.org/10.1016/j.chom.2016.05.012</a>
</div>
<div id="ref-benedictow2004BlackDeath13461353" class="csl-entry" role="doc-biblioentry">
9. Benedictow, O. J. (2004). <em>The <span>Black Death</span>, 1346-1353: The <span>Complete History</span></em>. <span>Boydell Press</span>.
</div>
<div id="ref-varlik2020PlagueThatNever" class="csl-entry" role="doc-biblioentry">
10. Varlık, N. (2020). The plague that never left: Restoring the <span>Second Pandemic</span> to <span>Ottoman</span> and <span>Turkish</span> history in the time of <span>COVID</span>-19. <em>New Perspectives on Turkey</em>, <em>63</em>, 176–189. <a href="https://doi.org/10.1017/npt.2020.27">https://doi.org/10.1017/npt.2020.27</a>
</div>
<div id="ref-morelli2010YersiniaPestisGenome" class="csl-entry" role="doc-biblioentry">
11. Morelli, G., Song, Y., Mazzoni, C. J., Eppinger, M., Roumagnac, P., Wagner, D. M., Feldkamp, M., Kusecek, B., Vogler, A. J., Li, Y., Cui, Y., Thomson, N. R., Jombart, T., Leblois, R., Lichtner, P., Rahalison, L., Petersen, J. M., Balloux, F., Keim, P., … Achtman, M. (2010). <span><em>Yersinia Pestis</em></span> genome sequencing identifies patterns of global phylogenetic diversity. <em>Nature Genetics</em>, <em>42</em>(12), 1140–1143. <a href="https://doi.org/10.1038/ng.705">https://doi.org/10.1038/ng.705</a>
</div>
<div id="ref-pisarenko2021YersiniaPestisStrains" class="csl-entry" role="doc-biblioentry">
12. Pisarenko, S. V., Evchenko, A. Yu., Kovalev, D. A., Evchenko, Y., Bobrysheva, O. V., Shapakov, N. A., Volynkina, A. S., &amp; Kulichenko, A. N. (2021). Yersinia pestis strains isolated in natural plague foci of <span>Caucasus</span> and <span>Transcaucasia</span> in the context of the global evolution of species. <em>Genomics</em>, <em>113</em>(4), 1952–1961. <a href="https://doi.org/10.1016/j.ygeno.2021.04.021">https://doi.org/10.1016/j.ygeno.2021.04.021</a>
</div>
<div id="ref-xu2014WetClimateTransportation" class="csl-entry" role="doc-biblioentry">
13. Xu, L., Stige, L. Chr., Kausrud, K. L., Ben Ari, T., Wang, S., Fang, X., Schmid, B. V., Liu, Q., Stenseth, N. Chr., &amp; Zhang, Z. (2014). Wet climate and transportation routes accelerate spread of human plague. <em>Proceedings of the Royal Society B: Biological Sciences</em>, <em>281</em>(1780), 20133159. <a href="https://doi.org/10.1098/rspb.2013.3159">https://doi.org/10.1098/rspb.2013.3159</a>
</div>
<div id="ref-tan2002AtlasPlagueIts" class="csl-entry" role="doc-biblioentry">
14. Tan, J., Liu, Y., Shen, E., Zhu, W., Wang, W., Li, R., &amp; Yang, L. (2002). <span>Towards the atlas of plague and its environment in the People’s Republic of China: idea, principle and methodology of design and research results</span>. <em>Huan Jing Ke Xue= Huanjing Kexue</em>, <em>23</em>(3), 1–8.
</div>
<div id="ref-kahan1979SocialAspectsPlague" class="csl-entry" role="doc-biblioentry">
15. Kahan, A. (1979). Social <span>Aspects</span> of the <span>Plague Epidemics</span> in <span>Eighteenth</span>-<span>Century Russia</span>. <em>Economic Development and Cultural Change</em>, <em>27</em>(2), 255–266.
</div>
<div id="ref-hashemishahraki2016PlagueIranIts" class="csl-entry" role="doc-biblioentry">
16. Hashemi Shahraki, A., Carniel, E., &amp; Mostafavi, E. (2016). Plague in <span>Iran</span>: Its history and current status. <em>Epidemiology and Health</em>, <em>38</em>, e2016033. <a href="https://doi.org/10.4178/epih.e2016033">https://doi.org/10.4178/epih.e2016033</a>
</div>
<div id="ref-eroshenko2019CirculationYersiniaPestis" class="csl-entry" role="doc-biblioentry">
17. Eroshenko, G. A., Popov, N. V., Alkhova, Zh. V., Balykova, A. N., Kukleva, L. M., Chervyakova, N. S., Maykanov, N. S., Sarmuldina, A. Kh., &amp; Kutyrev, V. V. (2019). Circulation of <span><em>Yersinia</em></span><span> <em>Pestis</em></span> in the <span>Volga</span>-<span>Ural</span> sandy focus: Spatiotemporal analysis. <em>Problems of Particularly Dangerous Infections</em>, <em>3</em>, 51–57. <a href="https://doi.org/10.21055/0370-1069-2019-3-51-57">https://doi.org/10.21055/0370-1069-2019-3-51-57</a>
</div>
<div id="ref-munyenyiwa2019PlagueZimbabwe1974" class="csl-entry" role="doc-biblioentry">
18. Munyenyiwa, A., Zimba, M., Nhiwatiwa, T., &amp; Barson, M. (2019). Plague in <span>Zimbabwe</span> from 1974 to 2018: A review article. <em>PLOS Neglected Tropical Diseases</em>, <em>13</em>(11), e0007761. <a href="https://doi.org/10.1371/journal.pntd.0007761">https://doi.org/10.1371/journal.pntd.0007761</a>
</div>
<div id="ref-green2018PuttingAfricaBlack" class="csl-entry" role="doc-biblioentry">
19. Green, M. H. (2018). Putting <span>Africa</span> on the <span>Black Death</span> map: Narratives from genetics and history. <em>Afriques. Débats, Méthodes Et Terrains d’histoire</em>, <em>09</em>. <a href="https://doi.org/10.4000/afriques.2125">https://doi.org/10.4000/afriques.2125</a>
</div>
</div>
</section>

<section>
<section id="supplementary-slides" class="title-slide slide level2">
<h2>Supplementary Slides</h2>

</section>
<section id="branch-length-distribution" class="slide level3">
<h3>Branch Length Distribution</h3>
</section>
<section id="phylogeography" class="slide level3">
<h3>Phylogeography</h3>
<hr>
<aside class="notes">
<ul>
<li>A lot of uncertainty about the geographic origins and spread of the Second Pandemic (genetic data).</li>
<li>Short sampling time frame (ex. Black Death), really hard to get the root position correct without prior information. Also Germany, Poland, Lithuania.</li>
<li>Sparse geographic sampling. 500 years, 40 genomes, 11 countries.</li>
<li>To reconstruct spread (point migrations), need to pin Yp location to single points (country).</li>
<li>Intra-continental spread reconstructions have no low confidence.</li>
<li>Companion is the phylogeographic threshold BETS test with geography instead of dates? Randomize the countries.</li>
<li>BETS test, all dates set to contemporaneous (or randomized)?</li>
</ul>
</aside>
<hr>
<figure>
<img data-src="1.PRE_timetree.svg" id="fig:1.PRE_timetree" height="500" alt="Figure 13: Phylogeography of the Second Plague Pandemic (1.PRE)." /><figcaption aria-hidden="true">Figure 13: Phylogeography of the Second Plague Pandemic (1.PRE).</figcaption>
</figure>
</section>
<section id="phylogeography-1" class="slide level3">
<h3>Phylogeography</h3>
<hr>
<blockquote>
<p>Inter-country geogrpahy….</p>
</blockquote>
<figure>
<img data-src="1.PRE_scatter.svg" id="fig:1.PRE_scatter" height="500" alt="Figure 14: Phylogeography of the Second Plague Pandemic (1.PRE). Blue diamonds indicate nodes where the ancestral location was inferred to be in a single country with a probability greater than 0.95 ." /><figcaption aria-hidden="true">Figure 14: Phylogeography of the Second Plague Pandemic (1.PRE). Blue diamonds indicate nodes where the ancestral location was inferred to be in a single country with a probability greater than 0.95 .</figcaption>
</figure>
</section>
<section id="notes" class="slide level3">
<h3>Notes</h3>
<ul>
<li>Confidence Interval Distribution</li>
</ul>
</section></section>
    </div>
  </div>

  <script src="https://unpkg.com/reveal.js@^4//dist/reveal.js"></script>

  // remote reveal.js plugins
  <script src="https://unpkg.com/reveal.js@^4//plugin/notes/notes.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/search/search.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/zoom/zoom.js"></script>
  // local reveal.js plugins  
  <script src="pandoc/plugins/reveal.js-menu/menu.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
      
        // Display a presentation progress bar
        progress: true,
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,

        // reveal.js plugins
        plugins: [
          RevealNotes,
          RevealSearch,
          RevealZoom,
          RevealMenu
        ]
      });
    </script>
    </body>
</html>
