---
title: Plagued by a cryptic clock
subtitle: Insight and issues from the global phylogeny of _Yersinia pestis_
title-img: "https://rawcdn.githack.com/ktmeaton/plague-phylogeography/e74d7b0d7\
  32023d109e1abbefde3305eae589ebd/docs/images/thumbnail_DHSI2020.png"
author:
  - [[Katherine Eaton\|Eaton, K.]], [[Leo Featherstone\|Featherstone, L.]], [[Sebastian\|Duchene, S.]], [[Ann Carmichael\|Carmichael, A.]], [[N√ºkhet Varlƒ±k\|Varlƒ±k, N.]], [[Brian Golding\|Golding, G. B.]], [[Edward Holmes\|Holmes, E.]], [[Hendrik Poinar\|Poinar, H. N.]]
venue: "BEAP"
progress: true
slideNumber: true
slideLevel: 3
tags:
  - üìù/üå±
  - üß® 
status: priority
type: [[Slides]]
due: 2021-10-01
time: 11:00
date: 2021 October 01
toc: false
tblPrefix: Table
figPrefix:
compile-revealjs: "conda activate pandoc && pandoc/convert_wikilinks.py --input 'BEAP 2021-10-01.md' --output 'BEAP 2021-10-01_convert.md' && pandoc --standalone -o 'BEAP 2021-10-01.html' -t revealjs --slide-level 3 --template pandoc/templates/revealjs-obsidian/template.html --css pandoc/templates/revealjs-obsidian/beap_2021-10-01.css --filter pandoc-crossref --citeproc --bibliography pandoc/bib/library.bib --csl pandoc/csl/apa-numeric-superscript-brackets.csl 'BEAP 2021-10-01_convert.md' && rm 'BEAP 2021-10-01_convert.md';"
---

## Introduction 

### Disease Exposure and Disease Experience

### Zoonoses

### Plague

### {data-transition="none"}


![
Timeline of plague based on historical evidence.
](test.svg){#fig:test}

### {data-transition="none"}


<aside class="notes">
- This is a timeline of plague based on historical evidence.<br>
- White: Periods with written evidence of plague.<br>
- Earliest textual evidence of plague in Late Antiquity. 3 historical catalogues document a pandemic from the 6th - 8th century. Dubbed the First Pandemic.<br>
- Plague reappears in the historical record for a second time, 14th-19th century, Second Pandemic.<br>
- Finally, appears for a third time, 19th-20th century
</aside>

![
Timeline of plague based on historical evidence.
](https://rawcdn.githack.com/ktmeaton/obsidian-public/a7bdda5/academic/plague_timeline_blank.svg){#fig:plague_timeline_blank height=500px}

### {data-transition="none"}

![
Timeline of plague based on _Y. pestis_ genomes from the 20<sup>th</sup> to 21<sup>st</sup> centuries CE.
](https://rawcdn.githack.com/ktmeaton/obsidian-public/a7bdda5/academic/plague_timeline_modern.svg){#fig:plague_timeline_modern height=500px}

### {data-transition="none"}

![
Timeline of plague based on _Y. pestis_ genomes from the 3<sup>rd</sup> millenium BCE to the 21<sup>st</sup> century CE.
](https://rawcdn.githack.com/ktmeaton/obsidian-public/a7bdda5/academic/plague_timeline_ancient.svg){#fig:plague_timeline_ancient height=500px}

### Objectives


## _Y. pestis_  Population Structure

### Population Structure

<hr> 

![
The maximum-likelihood phylogeny of 601 _Yersinia pestis_ genomes.
](divtree.svg){#fig:divtree}

### Global Molecular Clock

<hr>

<aside class="notes">
- Bayesian (BEAST) analysis was inconclusive (strict or relaxed).<br>
- Poor sampling, lack of convergence, low ESS.<br>
- Why? Too much rate variation!<br>
- Reducing the number of samples, reducing tip date uncertainty, fixing the tree topology.<br>
- Rates and dates are not informative!<br>
- Pictures of Sebastian and Leo.<br>
</aside>

> The global diversity of _Y. pestis_ is poorly modeled by a single molecular clock.

![
Root-to-tip regression
](rtt.svg){#fig:rtt height=400px}


### Population Molecular Clocks

<hr>

> Population clocks recover temporal signal in 9 of 12 populations.<br>

- Clock parameters: ESS scores > 200
- Relaxed clock (Uncorrelated lognormal) outperforms strict clocks.

## Observation ?:<br>Temporal Signal With aDNA

### Temporal Signal With aDNA

<aside class="notes">
- Second Pandemic (1.PRE) starts with a polytomy, root position is incorrectly inferred.
</aside>
	
<hr>

- All (3) ancient _Y. pestis_ populations had temporal signal.
- Bronze Age (0.PRE), First Pandemic (0.ANT4), Second Pandemic (1.PRE).

### Phylogeography

![Phylogeography of the Second Plague Pandemic (1.PRE). Grey bars around internal nodes define dating uncertainty, and represent the 95% highest posterior density (HPD) interval. Asterisks indicate clade separations with posterior support greater than 0.95. Blue diamonds indicate nodes where the ancestral location was inferred to be in a single country with a probability greater than 0.95 .](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/27404bd/main/auspice/all/chromosome/full/filter5/beast/geo/1.PRE_timetree.png){#fig:1.PRE_timetree width=100%}

## Observation ?:<br>Temporal Signal Without aDNA

### Temporal Signal Without aDNA

<aside class="notes">
- Uncertainty always increases further back in time, but for 2.MED uncertainty gets really big really fast.
- Insert time trees here?
- High uncertainty in the root position, leading to large confidence intervals.
- 1.ORI: 1806 - 1901
- 2.MED: 1443 - 1878
- Use the branching order in the maximum likelihood phylogeny to help root it?
- [@morelli2010YersiniaPestisGenome]<br>[@cui2013HistoricalVariationsMutation]<br>This Study
</aside>

<hr>

- 2 modern populations had temporal signal, despite having no ancient DNA calibrations.
- The Third Pandemic (1.ORI)
- *Medievalis* (2.MED), once thought to be the cause of the Black Death.

<br>

| Population |      Name      |    tMRCA (Lower)     |    tMRCA (Upper)     | Study |
|:----------:|:--------------:|:--------------------:|:--------------------:|:-----:|
|   1.ORI    | Third Pandemic | -381<br>1735<br>1806 | 1738<br>1863<br>1901 |  TBD  |
|   2.MED    |  *Medievalis*  | -638<br>1450<br>1560 | 1715<br>1750<br>1845 |  TBD  | 
	
Table: Table test! {#tbl:test}


## Observation ?:<br> No Temporal Signal

### Observation ?: No Temporal Signal

<hr>

<aside class="notes">
- 3 populations had insufficient phylogenetic and temporal information.<br>
</aside>

- Three populations have no temporal signal (Figure @fig:no_temporal_signal).
- Shortest sampling time frames.
- Node density is highest close to the root.

> These populations have rates and dates that are **non-informative**.<br>
> The **phylodynamic threshold** of _Y. pestis_ is greater than 50 years. <br>
> Comparisons over shorter time scales will have **limited resolving power**.

 ![No temporal signal](no_temporal_signal.svg){#fig:no_temporal_signal}

### Epidemic Analysis
<hr>

<aside class="notes">
> **Solution:** Other genetic markers?
</aside>

> Intra-epidemic diversity is poorly resolved by a core genome SNP phylogeny.<br>

- Black Death (1346 - 1353) _Y. pestis_ isolates are indistinguishable clones [@spyrou2016HistoricalPestisGenomes].

 ![Spread map of the Black Death](Map Black Death.png){#fig:map_black_death height=300px}

### The Third Pandemic

<hr>

<aside class="notes">
- **Solution:** Increased sampling of long internal branches.
</aside>

> ... of the Third Pandemic (1.ORI)...<br>

 ![Third Pandemic Origins](divtree_1.IN_1.ORI.svg){#fig:divtree_1.IN_1.ORI height=350px}

## Observation 2:<br> Node Date Conflicts

### Observation 2: Node Date Conflicts

<hr>

- Several populations have conflicting dates for their tMRCA.
- These populations are not monophyletic.
- Ex. *Antiqua* (0.ANT) is ancestral to the First Pandemic (0.ANT4).
- The tMRCA of *Antiqua* (0.ANT) incorrectly post-dates the First Pandemic (0.ANT4).

 ![Date Conflict](date_conflict.svg){#fig:date_conflict height=400px}

### Observation 2: Node Date Conflicts

<hr>

- Node density is low close to the root.
- Node dates are underestimated (too young). Can be off by a 1000 years!

> Insufficient sample sizes (1.ANT).<br>
> Insufficient geographic sampling (0.ANT).<br>
> Extensive rate variation (0.PE) with no ancient DNA calibrations.

 ![Suspicious Dates](suspicious_dates.svg){#fig:suspicious_dates}

### Arrival of Plague in Africa

<aside class="notes">
- Plague in Africa is really important for public health research = high interest.
- But the genetic evidence is very flimsy for estimating a continental tMRCA.
</aside>

<hr>

- Africa accounts for more than 90% of all human plague cases [@munyenyiwa2019PlagueZimbabwe1974] .
- But only 1.5% of _Y. pestis_ genomes are from Africa.
- Heavily debated when plague first arrived in Central and East Africa (1.ANT):
	- 4,964 BCE - 1322 CE (N=2) [@morelli2010YersiniaPestisGenome]
	- 1377 CE - 1650 CE (N=2) [@cui2013HistoricalVariationsMutation]
	- 1655 CE - 1835 CE (N=4)<sup>[This study]</sup>
- None of these dates overlap!!

> Genetic dates for the tMRCA in Africa are non-informative!<br>
> They should not be used as a basis for historical or archaeological interpretation [@green2018PuttingAfricaBlack].


## Observation 3:<br>???

### Observation 3: ??

> Recently emerged populations can be calibrated with early 20<sup>th</sup> century isolates.


## Conclusion

- Methodological and epidemiological findings.

## References
