---
title: Plagued by a cryptic clock
subtitle: Insight and issues from the global phylogeny of _Yersinia pestis_
title-img: "https://rawcdn.githack.com/ktmeaton/plague-phylogeography/e74d7b0d7\
  32023d109e1abbefde3305eae589ebd/docs/images/thumbnail_DHSI2020.png"
author:
  - [[Katherine Eaton\|Eaton, K.]], [[Leo Featherstone\|Featherstone, L.]], [[Sebastian\|Duchene, S.]], [[Ann Carmichael\|Carmichael, A.]], [[N√ºkhet Varlƒ±k\|Varlƒ±k, N.]], [[Brian Golding\|Golding, G. B.]], [[Edward Holmes\|Holmes, E.]], [[Hendrik Poinar\|Poinar, H. N.]]
venue: "BEAP"
progress: true
slideNumber: true
slideLevel: 3
tags:
  - üìù/üå±
  - üß® 
status: priority
type: [[Slides]]
due: 2021-10-01
time: 11:00
date: 2021 October 01
toc: false
tblPrefix: Table
figPrefix:
compile-revealjs: "conda activate pandoc && pandoc/convert_wikilinks.py --input 'BEAP 2021-10-01.md' --output 'BEAP 2021-10-01_convert.md' && pandoc --standalone -o 'BEAP 2021-10-01.html' -t revealjs --slide-level 3 --template pandoc/templates/revealjs-obsidian/template.html --css pandoc/templates/revealjs-obsidian/beap_2021-10-01.css --filter pandoc-crossref --citeproc --bibliography pandoc/bib/library.bib --csl pandoc/csl/apa-numeric-superscript-brackets.csl 'BEAP 2021-10-01_convert.md' && rm 'BEAP 2021-10-01_convert.md';"
---

## Introduction 

### Concept 1. Disease Exposure

<hr>

- How do we contract an infectious disease?
- How do we spread it?
- Why does this vary across time and space?

### Concept 2. Disease Experience

<hr>

- Are the symptoms mild?
- Does it become endemic and part of our everyday life?
- Is there catastrophic mortality?
- Does it radically change society?

### Zoonoses

<hr>

- Diseases that spread from non-human animals to humans.
- Enigmatic epidemiology that is often poorly understood.
	- COVID-19
	- Malaria
	- Influenza
	- Brucellosis
	- Lyme Disease
	- Plague

### Plague

<hr>

1. **Prolific presence in human populations, at least since the Bronze Age (~3000 YBP).**

![Geographic distribution of human plague cases (1772 - 2014) [@xu2019HistoricalGenomicData]. ](xu2019HistoricalGenomicData_fig1.jpg){#fig:xu_2019}

### Plague

<hr>

2. **Unnerving virulence and mortality.**
	- The medieval Black Death (1346-1353) killed more than 50% of Europe's population. [@benedictow2021CompleteHistoryBlack].
	- Untreated pneumonic plague has a case fatality rate of nearly 100% [@prentice2007Plague; @salam2020DeathsAssociatedPneumonic].

3. **It keeps coming back!**
	- First Pandemic (6<sup>th</sup> - 8<sup>th</sup> Century CE)
	- Second Pandemic (14<sup>th</sup> - 19<sup>th</sup> Century CE)
	- Third Pandemic (19<sup>th</sup> - 20<sup>th</sup> Century CE)

4. **A cryptic clock.**
	- Debated whether molecular clock techniques can produce informative rates and dates.
	- An extremely slow evolutionary rate coupled with extreme rate variation.

### Objectives

1. Is there any way to **estimate robust rates and dates** for _Y. pestis_?
2.  Do I have **anything new to say** about the history and epidemiology of plague?

## Materials and Methods

### Materials and Methods

<hr>

<aside class="notes">
- Bayesian (BEAST) analysis was inconclusive (strict or relaxed).<br>
- Poor sampling, lack of convergence, low ESS.<br>
- Why? Too much rate variation!<br>
- Reducing the number of samples, reducing tip date uncertainty, fixing the tree topology.<br>
- Rates and dates are not informative!<br>
- Pictures of Sebastian and Leo.<br>
</aside>

1. **601 publicly available _Y. pestis_ genomes.**
	- 540 modern (89.9%): 20<sup>th</sup> to 21<sup>st</sup> century CE.
	- 61 ancient (10.1%): 3<sup>rd</sup> millenium BCE to 18<sup>th</sup> century CE.
2. **Multiple alignment of the core genome.**
3. **Estimated a maximum-likelihood phylogeny.**
4. **Bayesian Evaluation of Temporal Signal (BETS).**
	- Strict Clock
	- Relaxed Clock
	- Global Clock
	- Population Clocks [@cui2013HistoricalVariationsMutation]
5. **Estimated rates and dates for populations with temporal signal.**

### Population Structure

<hr> 

![
The maximum-likelihood phylogeny of 601 _Yersinia pestis_ genomes.
](divtree.svg){#fig:divtree}

### Molecular Clocks

<hr> 

- The global clock analysis (all samples) was VERY unstable.

![
Root-to-tip regression of 601 _Yersinia pestis_ genomes using the maximum-likelihood phylogeny.
](rtt.svg){#fig:rtt height=400px}

### Molecular Clocks

<hr>

- Tried reducing sources of rate variation:
	- Decreasing the number of samples.
	- Fixing tip dates.
	- Fixing tree topology.
	
> The global diversity of _Y. pestis_ is poorly modeled by a single molecular clock.

<hr>

- Population clocks **recovered temporal signal** in 9 out of 12 populations.
- For all populations, the relaxed clock outerperformed the strict clock.


## The Phylodynamic Threshold

<aside class="notes">
- SARS-CoV-2: Is a virus, thus fast-evolving, threshold reached in 2 months of the start of the pandemic.
</aside>

> _"The point in time at which sufficient molecular evolutionary change has accumulated in available genome samples to obtain robust phylodynamic estimates."_<br>
> <br>
> _"Before the phylodynamic threshold is reached, genomic variation is so low that even large amounts of genome sequences may be insufficient to estimate the virus‚Äôs evolutionary rate and the time scale of an outbreak."_<br>
> <br>
> - Duchene et al. (2020) [@duchene2020TemporalSignalPhylodynamic].

### Rates of Evolutionary Change {data-transition="none"}

<hr>

- _Y. pestis_ has one of the slowest substitution rates among bacterial pathogens [@duchene2016GenomescaleRatesEvolutionary].
- _Y. pestis_ has substantial rate variation [@cui2013HistoricalVariationsMutation].

![Substitution rates across the maximum-clade credibility (MCC) trees.](rates_mcc_tree.svg){#fig:rates_mcc_tree height=400px}

### Rates of Evolutionary Change {data-transition="none"}

<hr>

- _Y. pestis_ has one of the slowest substitution rates among bacterial pathogens [@duchene2016GenomescaleRatesEvolutionary].
- _Y. pestis_ has substantial rate variation [@cui2013HistoricalVariationsMutation].

![Substitution rates across the maximum-clade credibility (MCC) trees with outliers removed.](rates_mcc_tree_no_outliers.svg){#fig:rates_mcc_tree_no_outliers height=400px}

### Why Do We Care?


- Rates and dates of short-term events will be meaningless.
	- Ex. Epidemics, recently emerged populations.
<br>

> But what does "short-term" mean?


### No Temporal Signal

<hr>

<aside class="notes">
- 3 populations had insufficient phylogenetic and temporal information.<br>
</aside>

- Three populations have no temporal signal (Figure @fig:no_temporal_signal).
- Shortest sampling time frames.
- Node density is highest close to the root.

> These populations have rates and dates that are **non-informative**.<br>
> The **phylodynamic threshold** of _Y. pestis_ is greater than 50 years. <br>
> Comparisons over shorter time scales will have **limited resolving power**.

![Maximum-likelihood phylogenies of _Yersinia pestis_ populations with no detectable temporal signal. Grey branches indicate outliers.](no_temporal_signal.svg){#fig:no_temporal_signal}

### Epidemic Analysis
<hr>

<aside class="notes">
> **Solution:** Other genetic markers?
</aside>

> Intra-epidemic diversity is poorly resolved by a core genome SNP phylogeny

- Black Death (1346 - 1353) _Y. pestis_ isolates are indistinguishable clones [@spyrou2016HistoricalPestisGenomes].

 ![Spread map of the Black Death [@benedictow2004BlackDeath13461353].](Map Black Death.png){#fig:map_black_death height=300px}

### The Third Pandemic

<hr>

<aside class="notes">
- **Solution:** Increased sampling of long internal branches.
</aside>

- The events leading up to the Third Pandemic (1.ORI) are highly uncertain.<br>
- When and how, did the epidemiology of plague change from localized epidemics in Yunnan Province, China to a global pandemic?

 ![Third Pandemic Origins](divtree_1.IN_1.ORI.svg){#fig:divtree_1.IN_1.ORI height=350px}
 
## Ancient DNA

### Temporal Signal With Ancient DNA

<aside class="notes">
- Second Pandemic (1.PRE) starts with a polytomy, root position is incorrectly inferred.
</aside>
	
<hr>

All (3) ancient _Y. pestis_ populations had temporal signal:


- 1.PRE | Second Pandemic (14<sup>th</sup> - 19 <sup>th</sup> century CE)
- 0.ANT4 | First Pandemic (6<sup>th</sup> - 8 <sup>th</sup> century CE)
- 0.PRE | Bronze Age (3<sup>rd</sup> - 1<sup>st</sup> Millennia BCE)

 ![Maximum-likelihood phylogenies of ancient _Y. pestis_ populations with detectable temporal signal. Grey branches indicate outliers.](temporal_signal_ancient.svg){#fig:temporal_signal_ancient}
 
### What Can We Learn?

<aside class="notes">
- Example: What if we tried to reconstruct COVID pandemic just from European data? 2020, Italy.
</aside>

<hr>

- Informative rates and dates can be obtained when **the sampling time frame is >500 years**.
- Node age **uncertainty can be 100+ years** (95% HPD).
- Genetic evidence suggests that the **plague pandemics may have earlier origins**.
- Most evidence of historical plague comes from European sources.
- New scholarship of non-European sources is rewriting these narratives [@varlik2020PlagueThatNever].

 <br>
 
| Population |      Event      | tMRCA (Lower) | tMRCA (Upper) | Historical Start |
|:----------:|:---------------:|:------------:|:-------------:|:----------------:|
|   1.PRE    | Second Pandemic |     1214     |     1315      |       1346       |
|   0.ANT4   | First Pandemic  |     272      |      466      |       541        |
|   0.PRE    |   Bronze Age    |    -3098     |     -2786     |        --        |

Table: Node dating of key historical plague events. {#tbl:node_dating_ancient}
 
## Modern DNA

### Temporal Signal With Modern DNA

<aside class="notes">
- Uncertainty always increases further back in time, but for 2.MED uncertainty gets really big really fast.
- Insert time trees here?
- High uncertainty in the root position, leading to large confidence intervals.
- Use the branching order in the maximum likelihood phylogeny to help root it?
- [@morelli2010YersiniaPestisGenome]<br>[@cui2013HistoricalVariationsMutation]<br>This Study
- 3rd pandemic date originally 1894, pushed back to 1855. Some argue as early as 1722.
</aside>

<hr>

- Two modern populations had temporal signal, despite having no ancient DNA calibrations.
	- The Third Pandemic (1.ORI), global.
	- *Medievalis* (2.MED), Asia.
- Have high epidemiological significance:
	- High mortality epidemics.
	- Extensive and rapid geographic spread.

 ![Maximum-likelihood phylogenies of modern _Y. pestis_ populations with detectable temporal signal. Grey branches indicate outliers.](temporal_signal_modern.svg){#fig:temporal_signal_modern}

### Temporal Signal With Modern DNA

<hr>

- As more data becomes available, the **node dates are becoming younger** [@pisarenko2021YersiniaPestisStrains].

<br>

|  Population   |          Name          | Study |       Samples       |        tMRCA (Lower)         |        tMRCA (Upper)         |
|:-------------:|:----------------------:|:-----:|:-------------------:|:----------------------------:|:----------------------------:|
| <br>1.ORI<br> | <br>Third Pandemic<br> |  TBD  | 6<br>17<br>93<br>117 | -381<br>1735<br>1742<br>1806 | 1738<br>1863<br>1842<br>1901 | 
| <br>2.MED<br> |  <br>*Medievalis*<br>  |  TBD  | 2<br>25<br>68<br>116 | -638<br>1450<br>1298<br>1560 | 1715<br>1750<br>1582<br>1845 |
	
Table: Node dating of modern _Yersinia pestis_ populations. {#tbl:node_dating_modern}

### Timeline of The Third Pandemic

<aside class="notes">
	 ![Third Pandemic Origins](divtree_1.IN_1.ORI.svg){height=350px}
</aside>

<hr>
	
- We estimate that the Third Pandemic (1.ORI) **emerged between 1806 and 1901 (1885)**.
- This **aligns with historical evidence** from Chinese epidemiologists [@xu2014WetClimateTransportation] [@tan2002AtlasPlagueIts].
- 1772 - 1800:  Human plague cases are isolated to Yunnan, China.
- 1880 - 1900: Plague spreads to the south-eastern provinces.
- 1894: Plague spreads worldwide out of Hong Kong.

|                                                                |                                                                    | 
|:--------------------------------------------------------------:|:------------------------------------------------------------------:|
| ![Third Pandemic Origins](divtree_1.IN_1.ORI.svg){width=500px} | ![Third Pandemic Origins](third_pandemic_origins.png){width=300px} |

### *Medievalis*

<hr>

- We estimate that *Medievalis* (2.MED) **emerged between 1560 and 1845 (1796)**.
- This **aligns with historical evidence** [@kahan1979SocialAspectsPlague; @hashemishahraki2016PlagueIranIts; @eroshenko2019CirculationYersiniaPestis].
- Late 1798: Human plague cases are observed in the Caucasus.
- 1800s: Plague cases throughout the Caspian Sea region.
- 1900s-2000s: Widespread dispersal across Asia.

![The geographic distribution of *Medievalis* (2.MED).](map_2.MED.svg)

### Whodunnit?

<hr>

- *Medievalis* (2.MED), *Pestoides* (0.PE), or the Second Pandemic (1.PRE)?

![](Map 0.PE 1.PRE 2.MED.png){height=400px}

## Good Temporal Signal<br>‚â†<br>Good Node Dates

### Node Date Conflicts

<hr>

- Several populations have conflicting dates for their tMRCA.
- These populations are not monophyletic.
- Ex. *Antiqua* (0.ANT) is ancestral to the First Pandemic (0.ANT4).
- The tMRCA of *Antiqua* (0.ANT) incorrectly post-dates the First Pandemic (0.ANT4).

 ![Node dating conflicts between the ancestral *Antiqua* (0.ANT) and the First Pandemic (0.ANT4).](date_conflict.svg){#fig:date_conflict height=400px}

### Node Date Conflicts

<hr>

- Node density is low close to the root.
- Node dates are underestimated (too young). Can be off by a 1000 years!

> Insufficient sample sizes (1.ANT).<br>
> Insufficient geographic sampling (0.ANT).<br>
> Extensive rate variation (0.PE) with no ancient DNA calibrations.

 ![Maximum-likelihood phylogenies of modern _Y. pestis_ populations lacking internal calibrations. Grey branches indicate outliers.](suspicious_dates.svg){#fig:suspicious_dates}

### Arrival of Plague in Africa

<aside class="notes">
- Plague in Africa is really important for public health research = high interest.
- But the genetic evidence is very flimsy for estimating a continental tMRCA.
</aside>

<hr>

- Africa accounts for more than 90% of all human plague cases [@munyenyiwa2019PlagueZimbabwe1974] .
- But only 1.5% of _Y. pestis_ genomes are from Africa.
- Heavily debated when plague first arrived in Central and East Africa (1.ANT).

<br>

|  Population   |           Name            | Study |     Samples     |     tMRCA (Lower)     |    tMRCA (Upper)     |
|:-------------:|:-------------------------:|:-----:|:---------------:|:---------------------:|:--------------------:|
| <br>1.ANT<br> | <br>African *Antiqua*<br> |  TBD  | 2<br>2<br>4<br> | -4964<br>1377<br>1655 | 1322<br>1650<br>1835 |
	
<br> 

> Genetic dates for the tMRCA of 1.ANT in Africa are non-informative!<br>
> They should not be used as a basis for historical or archaeological interpretation [@green2018PuttingAfricaBlack].



## Conclusions

### Methodology

<hr>
> **Poor performance of a global molecular clock.**<br>
> - Intensive global sampling  ‚Üí More genetic diversity ‚Üí More rate variation.<br>
> - Partitioning the data by population stabilizes clock estimates.

> **The phylodynamic threshold is very important!**<br>
> - A short sampling time frame (<50 years) led to non-informative rates and dates.<br>
> - Longer sampling time frames (>90 years) led to detectable temporal signal.

> **Temporal signal does not mean _all_ dates will be informative.**<br>
> - Insufficient sampling produces conflicting node dates.<br>
> - Can be detected in trees with few nodes close to the root.


### Plague History


> The tMRCAs of **ancient plague pandemics are older** than previously thought.<br>
> - Challenges Eurocentric views of  plague history.

> The tMRCAs of **modern plague are younger** than previously thought.<br>
> - Aligns with historical evidence.

## References

<div id="refs"></div>

## Supplementary Slides

### Branch Length Distribution

### Phylogeography

<hr>

<aside class="notes">
- A lot of uncertainty about the geographic origins and spread of the Second Pandemic (genetic data).
- Short sampling time frame (ex. Black Death), really hard to get the root position correct without prior information. Also Germany, Poland, Lithuania.
- Sparse geographic sampling. 500 years, 40 genomes, 11 countries.
- To reconstruct spread (point migrations), need to pin Yp location to single points (country).
- Intra-continental spread reconstructions have no low confidence.
- Companion is the phylogeographic threshold BETS test with geography instead of dates? Randomize the countries.
- BETS test, all dates set to contemporaneous (or randomized)?
</aside>

<hr>

![Phylogeography of the Second Plague Pandemic (1.PRE).](1.PRE_timetree.svg){#fig:1.PRE_timetree height=500px}
 
### Phylogeography
 
 <hr>
 
 > Inter-country geogrpahy....
 
![Phylogeography of the Second Plague Pandemic (1.PRE). Blue diamonds indicate nodes where the ancestral location was inferred to be in a single country with a probability greater than 0.95 .](1.PRE_scatter.svg){#fig:1.PRE_scatter height=500px}

### Notes

- Confidence Interval Distribution