% ------------------------------------------------------------
% Pandoc Default Begins

% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
\PassOptionsToPackage{dvipsnames,svgnames*,x11names*}{xcolor}
%
\documentclass[
]{report}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
	\usepackage[T1]{fontenc}
	\usepackage[utf8]{inputenc}
	\usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
	\usepackage{unicode-math}
	\defaultfontfeatures{Scale=MatchLowercase}
	\defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
	\usepackage[]{microtype}
	\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
	\IfFileExists{parskip.sty}{%
		\usepackage{parskip}
	}{% else
		\setlength{\parindent}{0pt}
		\setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
	\KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
	pdftitle={Big Data, Small Microbes: Genomic analysis of the plague bacterium Yersinia pestis},
	pdfauthor={Katherine Eaton},
	colorlinks=true,
	linkcolor=Maroon,
	filecolor=Maroon,
	citecolor=Blue,
	urlcolor=Blue,
	pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
	\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\makeatletter
\@ifpackageloaded{subfig}{}{\usepackage{subfig}}
\@ifpackageloaded{caption}{}{\usepackage{caption}}
\captionsetup[subfloat]{margin=0.5em}
\AtBeginDocument{%
\renewcommand*\figurename{Figure}
\renewcommand*\tablename{Table}
}
\AtBeginDocument{%
\renewcommand*\listfigurename{List of Figures}
\renewcommand*\listtablename{List of Tables}
}
\newcounter{pandoccrossref@subfigures@footnote@counter}
\newenvironment{pandoccrossrefsubfigures}{%
\setcounter{pandoccrossref@subfigures@footnote@counter}{0}
\begin{figure}\centering%
\gdef\global@pandoccrossref@subfigures@footnotes{}%
\DeclareRobustCommand{\footnote}[1]{\footnotemark%
\stepcounter{pandoccrossref@subfigures@footnote@counter}%
\ifx\global@pandoccrossref@subfigures@footnotes\empty%
\gdef\global@pandoccrossref@subfigures@footnotes{{##1}}%
\else%
\g@addto@macro\global@pandoccrossref@subfigures@footnotes{, {##1}}%
\fi}}%
{\end{figure}%
\addtocounter{footnote}{-\value{pandoccrossref@subfigures@footnote@counter}}
\@for\f:=\global@pandoccrossref@subfigures@footnotes\do{\stepcounter{footnote}\footnotetext{\f}}%
\gdef\global@pandoccrossref@subfigures@footnotes{}}
\@ifpackageloaded{float}{}{\usepackage{float}}
\floatstyle{ruled}
\@ifundefined{c@chapter}{\newfloat{codelisting}{h}{lop}}{\newfloat{codelisting}{h}{lop}[chapter]}
\floatname{codelisting}{Listing}
\newcommand*\listoflistings{\listof{codelisting}{List of Listings}}
\makeatother
 \newlength{\cslhangindent}
 \setlength{\cslhangindent}{1.5em}

\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}

\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
  {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
}%
{}

\title{Big Data, Small Microbes: Genomic analysis of the plague
bacterium \emph{Yersinia pestis}}
\author{Katherine Eaton}
\date{}

% Pandoc Default Ends
% ------------------------------------------------------------

\setlength{\parskip}{1em} % Paragraph indent
\setlength{\parindent}{2em} % Paragraph spacing
\captionsetup[figure]{font=small} % Shrink caption font size
\usepackage[nottoc,numbib]{tocbibind}



% ------------------------------------------------------------
% Packages
\usepackage{setspace} % doublespacing
\usepackage{fancyhdr} % page style fancy
\usepackage{calc,array} % table formatting and arraybackslash

% #############################################################################
% Document

\begin{document}


% -------------------------------------
% Setup
\hypersetup{pageanchor=false}
%\doublespacing
\pagenumbering{roman}
\pagestyle{plain}

% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Preface Pages

% -------------------------------------
% Half Title Page
\thispagestyle{empty}
\null\vfill
\begin{center}
	\LARGE \uppercase\expandafter{Big Data, Small Microbes}
\end{center}
\vfill
\vskip1in\newpage\setcounter{page}{1}

% -------------------------------------
% Full Title Page  
\thispagestyle{empty}
\null\vskip1in
\begin{center}
	\large\uppercase\expandafter{Big Data, Small Microbes: Genomic analysis
of the plague bacterium \emph{Yersinia pestis}}
\end{center}
\vfill
\begin{center}
	\rm\uppercase{By}\\
	\uppercase\expandafter{Katherine Eaton, B.A. (Hons)}\\
\end{center}\vskip.5in
\vfill
\begin{center}
	\sc   a thesis submitted to\\
		the Department of Anthropology\\
		and the school of graduate studies\\
		of mcmaster university\\
		in partial fulfilment of the requirements\\
		for the degree of\\
		Doctor of Philosophy
\end{center}
\vfill

\begin{center}
	\large
	\copyright\ Copyright\ by Katherine Eaton, \\
		All Rights Reserved
\end{center}
\vskip.5in
\newpage

% -------------------------------------
% Signature Page
\noindent
Doctor of Philosophy (2021) \hfill  McMaster University \\
(Department of
Anthropology)                                \hfill  Hamilton, Ontario, Canada


\vspace{1in}
\noindent
\begin{tabular}{ll}
TITLE:           & \parbox[t]{4in}{Big Data, Small Microbes: Genomic
analysis of the plague bacterium \emph{Yersinia pestis}} \\ \\
AUTHOR:          & \parbox[t]{4in}{Katherine Eaton \\ B.A. (Hons)
Anthropology, University of Alberta} \\ \\
SUPERVISOR:      & \parbox[t]{4in}{Dr.~Hendrik Poinar} \\ \\
NUMBER OF PAGES: & \parbox[t]{4in}{\pageref{NumPrefacePages}, \pageref{NumDocumentPages}}
\end{tabular}
\newpage

% -------------------------------------
% Lay Abstract
\chapter{Lay Abstract}
``A lay abstract of not more 150 words must be included explaining the
key goals and contributions of the thesis in lay terms that is
accessible to the general public.''

% -------------------------------------
% Full Abstract
\chapter{Abstract}
Abstract here (no more than 300 words).

% -------------------------------------
% Dedication
\newpage
\null\vfill
\begin{center}
\textsl{`You have to know the past to understand the present.' \\ - Carl
Sagan}
\end{center}
\vfill
	
% -------------------------------------
% Acknowledgements
\chapter{Acknowledgments}
Acknowledgments go here.
\newpage

% -------------------------------------
% Table of Contents
%\doublespacing
\tableofcontents

% -------------------------------------
% List of Figures
\newpage
\addvspace{10pt}
\let\saveaddvspace=\addvspace
\def\addvspace#1{}
\listoffigures
\let\addvspace=\saveaddvspace

% -------------------------------------
% List of Tables
\newpage
\addvspace{10pt}
\let\saveaddvspace=\addvspace
\def\addvspace#1{}
\listoftables
\let\addvspace=\saveaddvspace
\label{NumPrefacePages}
\newpage

% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Content Pages
\hypersetup{pageanchor=true}
\pagenumbering{arabic}
%\doublespacing
%\pagestyle{fancy}


\hypertarget{introduction}{%
\chapter{1 Introduction}\label{introduction}}

In 2011, I learned about a researcher named Dr.~Hendrik Poinar. His team
had just published a seminal paper, in which they identified the
causative agent of the infamous Black Death
(\protect\hyperlink{ref-bos2011DraftGenomeYersinia}{Bos et al., 2011}).
I discovered that this morbid term describes a pandemic that devastated
the world in the 14\textsuperscript{th} century, with unprecedented
mortality and spread. In less than 10 years (1346-1353) the Black Death
swept across Afro-Eurasia, killing 50\% of the population
(\protect\hyperlink{ref-cite}{\textbf{cite?}}). Outbreaks of this new
and mysterious disease, often referred to as \emph{the Plague},
reoccurred every 10 years on average
(\protect\hyperlink{ref-cite}{\textbf{cite?}}). This epidemic cycling
continued for 500 long years in Europe, but in Western Asia, the disease
never truly disappeared (\protect\hyperlink{ref-cite}{\textbf{cite?}}).
The 10-year window of the Black Death alone has an estimated global
mortality of 200 million people, making it the most fatal pandemic in
human history
(\protect\hyperlink{ref-sampath2021PandemicsThroughoutHistory}{Sampath
et al., 2021}), and it remains one of the most mysterious.

The cryptic nature of this medieval disease led to significant debate
among contemporaries. The dominant theory of contagion at the time was
\emph{miasma}, in which diseases were spread through noxious air
(\protect\hyperlink{ref-ober1982PlagueGranada13481349}{Ober \& Aloush,
1982}). Ibn al-Khatib, a notable Islamic scholar, took issue with this
theory. After studying outbreaks of \emph{Plague} in the
14\textsuperscript{th} century, he proposed an alternative hypothesis in
which \emph{minute bodies} were transmissible between humans
(\protect\hyperlink{ref-syed1981IslamicMedicine1000}{Syed, 1981}). Like
most controversial theories, this idea was not readily embraced. Some
400 years later, the British botanist Richard Bradley wrote a radical
treatise on \emph{Plague}
(\protect\hyperlink{ref-bradley1721PlagueMarseillesConsider}{Bradley,
1721}) where he similarly proposed that infectious diseases were caused
by living, microscopic agents. Again, this theory was rejected. It was
not until the 19\textsuperscript{th} century, with the experiments of
Louis Pasteur and Robert Koch, that this ``new'' perspective would
receive widespread acceptance
(\protect\hyperlink{ref-cite}{\textbf{cite?}}).

After it was established that \emph{a} living organism caused the Black
Death, the intuitive next step was to precisely identify \emph{the}
organism. The symptoms described in historical texts seem to incriminate
bubonic plague (\protect\hyperlink{ref-cite}{\textbf{cite?}}), a
bacterial pathogen that passes from \emph{rodents to humans}, and leads
to grotesquely swollen lymph nodes (buboes). On the other hand, the
rapid spread of the Black Death suggests this was a contagion primarily
driven by \emph{human to human} transmission, which more closely fits
the profile of an Ebola-like virus
(\protect\hyperlink{ref-scott2001BiologyPlaguesEvidence}{Scott \&
Duncan, 2001}). In the 2000s, geneticists began contributing novel
evidence to the debate, by retrieving pathogenic DNA from skeletal
remains (\protect\hyperlink{ref-cite}{\textbf{cite?}}). The plague
bacterium, \emph{Yersinia pestis}, played a central role in these
molecular investigations, as researchers sought to either establish or
refute its presence in medieval victims. The competitive nature of this
discourse fueled significant technological progress, and over the next
decade, the study of ancient DNA became a well-established discipline.
However, the origins of the Black Death remained unresolved, due to
numerous controversies surrounding DNA contamination and scientific
rigor (\protect\hyperlink{ref-cite}{\textbf{cite?}}).

As an undergraduate student of forensic anthropology, I was fascinated
by the rapid pace at which the field of ancient DNA was developing. I
attribute my developing academic obsession to two early-career
experiences. First, was reading the \emph{highly} entertaining
back-and-forth commentaries in academic journals, where plague
researchers would exchanges snide and personal insults
(\protect\hyperlink{ref-cite}{\textbf{cite?}}). It was clear that these
researchers cared \emph{deeply} about their work. Despite the toxicity,
I found these displays of passion to be engaging and refreshing, as
compared to the otherwise emotionally-sterile field of scientific
publishing.

The second defining experience, was the perplexing and often frustrating
task of trying to diagnose infectious diseases from skeletal remains. I
was intrigued by the idea of reconstructing an individual's life story
from their skeleton, and using this information to solve the
\emph{mysteries of the dead}. However, while some forms of trauma leave
diagnostic marks on bone (ex. sharp force), acute infectious diseases
rarely manifest in the skeleton
(\protect\hyperlink{ref-brown2013AnthropologyInfectiousDisease}{Brown \&
Inhorn, 2013};
\protect\hyperlink{ref-ortner2007DifferentialDiagnosisSkeletal}{Ortner,
2007}) and thus are `invisible' to an anthropologist. Because of this, I
found the new field of ancient DNA to be \emph{extremely} appealing, as
it offered a novel solution to this problem. Anthropologists could now
retrieve the \emph{precise pathogen} that had infected an individual,
and contribute new insight regarding disease exposure and experience
throughout human history. These experiences confirmed to me that
studying the ancient DNA of pathogens would be an exciting, dynamic, and
productive line of research for a graduate degree. I'm happy to say that
10 years later, I still agree with this statement, and by writing this
dissertation I hope to convince you, the reader, as well.

Which brings us back to Dr.~Hendrik Poinar and his team's seminal work
on the mysterious Black Death. The study, led by first author Kirsten
Bos, had found DNA evidence of the plague bacterium, \emph{Y. pestis},
in several Black Death victims buried in a mass grave in London
(\protect\hyperlink{ref-bos2011DraftGenomeYersinia}{Bos et al., 2011}).
But they did not just retrieve a few strands of DNA, they captured
millions of molecules (10.5 million to be precise) which allowed them to
reconstruct the entire \emph{Y. pestis} genome, comprising four million
DNA bases. The amount of molecular evidence was staggering, and offered
irrefutable proof that the plague bacterium was present during the time
of Black Death. Admittedly, any inference of causality would be mere
speculation, but given the extremely high case-fatality rates of plague
in modern times (\protect\hyperlink{ref-cite}{\textbf{cite?}}), there
was tentative support for the hypothesis that the causative agent of the
Black Death was plague.With a current sample size of N=1, additional
evidence was needed to strengthen this association.

In 2014, I had the delight and privilege of being accepted into the
graduate program at McMaster University under the supervision of
Dr.~Poinar. Alongside other members of the ``Plague Team'', I set about
the daunting task of screening the skeletal remains of more than 1000
individuals for molecular evidence of \emph{Y. pestis}. This material
was generously provided by collaborators, who were similarly invested in
the idea that ancient DNA could identify infectious diseases in their
sites. These archaeological remains reflected nearly a millennium of
human history, with sampling ages ranging from the 9\textsuperscript{th}
to the 19\textsuperscript{th} century CE. The geographic diversity was
also immense, with individuals sampled across Europe, Africa, and Asia.

Of the 1000+ individuals screened, approximately 30\% (N=298) of these
originated from Denmark. As a result, it should be unsurprising that we
had the greatest success in identifying ancient \emph{Y. pestis} in
Denmark. Over a period of 5 years, members of the ``Plague Team''
reconstructed full \emph{Y. pestis} genomes from 13 Danish individuals
dated to the Medieval and Early Modern period. In Chapter

\begin{itemize}
\item
  \hyperref[sec:introduction]{Some Displayed Text}
\item
  \nameref{sec:introduction}
\item
  \autoref[sec:introduction]
\end{itemize}

, to investigate these questions under the supervision of Dr.~Poinar at
the McMaster Ancient DNA Centre. Alongside other members of the ``Plague
Team'', we first set about the daunting task

In the following decade (2011-2021) more than 100 ancient \emph{Y.
pestis} genomes have been retrieved from skeletal remains across Europe,
making plague the \emph{most intensively sequenced historical disease}.
The sequencing of modern isolates has accelerated in tandem, with over
1000 genomes produced from 20\textsuperscript{th} and
21\textsuperscript{st} century plague outbreaks
(\protect\hyperlink{ref-cite}{\textbf{cite?}}). Because of this influx
of evidence, the research questions changed accordingly. Geneticists
were no longer interested in \emph{just} establishing the presence
\emph{Y. pestis} during the short time frame of the Black Death
(1346-1353), they wanted to know \emph{how it behaved} throughout the
long 500 years of this pandemic. These questions included lines of
inquiry such as: How did this ancient plague pandemic spread? Did it
take the rare form of pneumonic plague, where the bacterium could be
rapidly transmitted between humans along maritime routes? Or were
outbreaks highly localized cases of bubonic plague, where disease
exposure in humans occurs via spillover from nearby wild rodent
populations? And perhaps most importantly of all, why have we never
experienced another pandemic quite like the Black Death?

These nuanced epidemiological questions, particularly the last one, were
what motivated and resonated with me. In 2014, I had the delight and
privilege of being accepted into the graduate program at McMaster
University, to investigate these questions under the supervision of
Dr.~Poinar at the McMaster Ancient DNA Centre. Alongside other members
of the ``Plague Team'', we first set about the daunting task

Previously, I referred to one of the ultimate questions in plague
research: why have we never experienced another pandemic quite like the
Black Death?

In 2019, my relationship with infectious diseases transformed from an
intellectual curiosity to a lived experience. The emergence of the novel
coronavirus (SARS-CoV-2) triggered a global pandemic, operating on a
scale that had not been seen for a 100 years. For years, I had written
grants to fund my plague research under the auspice of \emph{we have to
know the past to understand the present}
(\protect\hyperlink{ref-cite}{\textbf{cite?}}).

\hypertarget{ncbimeta-efficient-and-comprehensive-metadata-retrieval-from-ncbi-databases}{%
\chapter{2 NCBImeta: Efficient and comprehensive metadata retrieval from
NCBI
databases}\label{ncbimeta-efficient-and-comprehensive-metadata-retrieval-from-ncbi-databases}}

\setlength{\parindent}{0em}

Published 03 February 2020 in\\
\emph{The Journal of Open Source Software}, 5(46), 1990.\\
\url{https://doi.org/10.21105/joss.01990}\strut \\
Licensed under a Creative Commons Attribution 4.0 International
License.\\
\hspace*{0.333em}

Katherine Eaton\textsuperscript{1,2}\\
\hspace*{0.333em}

\textsuperscript{1} McMaster Ancient DNA Centre, McMaster University\\
\textsuperscript{2} Department of Anthropology, McMaster University

\setlength{\parindent}{2em}

\hypertarget{plagued-by-a-cryptic-clock-insight-and-issues-from-the-global-phylogeny-of-yersinia-pestis}{%
\chapter{\texorpdfstring{3 Plagued by a cryptic clock: Insight and
issues from the global phylogeny of \emph{Yersinia
pestis}}{3 Plagued by a cryptic clock: Insight and issues from the global phylogeny of Yersinia pestis}}\label{plagued-by-a-cryptic-clock-insight-and-issues-from-the-global-phylogeny-of-yersinia-pestis}}

\setlength{\parindent}{0em}

Submitted 06 December 2021 to\\
\emph{Nature Communications}.\\
\url{https://www.researchsquare.com/article/rs-1146895}\strut \\
Licensed under a Creative Commons Attribution 4.0 International
License\\
\hspace*{0.333em}

Katherine Eaton\textsuperscript{1,2}, Leo
Featherstone\textsuperscript{3}, Sebastian Duchene\textsuperscript{3},
Ann G. Carmichael\textsuperscript{4}, Nükhet Varlık\textsuperscript{5},
G. Brian Golding\textsuperscript{6}, Edward C.
Holmes\textsuperscript{7}, Hendrik N.
Poinar\textsuperscript{1,2,8,9,10}\\
\hspace*{0.333em}

\textsuperscript{1}McMaster Ancient DNA Centre, McMaster University,
Hamilton, Canada.\\
\textsuperscript{2}Department of Anthropology, McMaster University,
Hamilton, Canada.\\
\textsuperscript{3}The Peter Doherty Institute for Infection and
Immunity, University of Melbourne, Melbourne, Australia.\\
\textsuperscript{4}Department of History, Indiana University
Bloomington, Bloomington, USA.\\
\textsuperscript{5}Department of History, Rutgers University-Newark,
Newark, USA.\\
\textsuperscript{6}Department of Biology, McMaster University, Hamilton,
Canada.\\
\textsuperscript{7}Sydney Institute for Infectious Diseases, School of
Life \& Environmental Sciences and School of Medical Sciences,
University of Sydney, Sydney, Australia.\\
\textsuperscript{8}Department of Biochemistry, McMaster University,
Hamilton, Canada.\\
\textsuperscript{9}Michael G. DeGroote Institute of Infectious Disease
Research, McMaster University, Hamilton, Canada.\\
\textsuperscript{10}Canadian Institute for Advanced Research, Toronto,
Canada.\\

\setlength{\parindent}{2em}

\hypertarget{plague-in-denmark-1000-1800-a-longitudinal-study-of-yersinia-pestis}{%
\chapter{\texorpdfstring{4 Plague in Denmark (1000-1800): A longitudinal
study of \emph{Yersinia
pestis}}{4 Plague in Denmark (1000-1800): A longitudinal study of Yersinia pestis}}\label{plague-in-denmark-1000-1800-a-longitudinal-study-of-yersinia-pestis}}

\setlength{\parindent}{0em}

Prepared 07 December 2021 for submission as a Brief Report to\\
\emph{The Proceedings of the National Academy of Sciences}\\
Licensed under a Creative Commons Attribution 4.0 International
License\\
\hspace*{0.333em}

Katherine Eaton\emph{\textsuperscript{1,2}, Ravneet
Sidhu}\textsuperscript{1,2}, Leo Featherstone\textsuperscript{3},
Sebastian Duchene\textsuperscript{3}, Ann G.
Carmichael\textsuperscript{4}, Nükhet Varlık\textsuperscript{5}, G.
Brian Golding\textsuperscript{6}, Hendrik N.
Poinar\textsuperscript{1,2,8,9,10}\\
\hspace*{0.333em}

*Contributed equally.

\textsuperscript{1}McMaster Ancient DNA Centre, McMaster University,
Hamilton, Canada.\\
\textsuperscript{2}Department of Anthropology, McMaster University,
Hamilton, Canada.\\
\textsuperscript{3}The Peter Doherty Institute for Infection and
Immunity, University of Melbourne, Melbourne, Australia.\\
\textsuperscript{4}Department of History, Indiana University
Bloomington, Bloomington, USA.\\
\textsuperscript{5}Department of History, Rutgers University-Newark,
Newark, USA.\\
\textsuperscript{6}Department of Biology, McMaster University, Hamilton,
Canada.\\
\textsuperscript{8}Department of Biochemistry, McMaster University,
Hamilton, Canada.\\
\textsuperscript{9}Michael G. DeGroote Institute of Infectious Disease
Research, McMaster University, Hamilton, Canada.\\
\textsuperscript{10}Canadian Institute for Advanced Research, Toronto,
Canada.\\
\hspace*{0.333em}

\setlength{\parindent}{2em}

\hypertarget{conclusion}{%
\chapter{5 Conclusion}\label{conclusion}}

As a paper on software development, its contributions and significance
to the field of anthropology are understandably unclear. I admittedly
targeted this article exclusively towards computational biologists
because, at the time, few anthropologists had expressed interested in
the issue of collecting and curating sequence data from online
repositories. However, since its publication, my software has been used
to support several bodies of anthropological research.

The database software NCBImeta was recently used to support an
environmental reconstruction of Beringia
(\protect\hyperlink{ref-murchieInPrepNoBonesIt}{Murchie et al., In
Prep}), the former land-bridge that facilitated early human migrations
into North America from northeast Asia. The study by Murchie et
al.~furthers our understanding of the peopling of the Americas, and the
possible interactions between early human populations and large animals
(ie. megafauna) before the Last Glacial Period (\textasciitilde12,000
years ago).

This tool was also recently used to curate sequence data in a case study
of the zoonotic disease brucellosis in the 14\textsuperscript{th}
century (\protect\hyperlink{ref-hiderInPrepExaminingPathogenDNA}{Hider
et al., In Prep}). The pioneering work by Hider et al.~demonstrates how
pathogen DNA preserves differently throughout the body, ranging from
being the dominant microorganism in several tissues while being
completely absent in others. It raises an important cautionary note for
ancient DNA analysis and the anthropology of disease, by empirically
demonstrating how sampling strategies can bias our understanding of what
diseases were present in past populations.

\hypertarget{bibliography}{%
\chapter*{References}\label{bibliography}}
\addcontentsline{toc}{chapter}{References}

\hypertarget{refs}{}
\begin{CSLReferences}{1}{0}
\leavevmode\vadjust pre{\hypertarget{ref-bos2011DraftGenomeYersinia}{}}%
Bos, K. I., Schuenemann, V. J., Golding, G. B., Burbano, H. A.,
Waglechner, N., Coombes, B. K., McPhee, J. B., DeWitte, S. N., Meyer,
M., Schmedes, S., Wood, J., Earn, D. J. D., Herring, D. A., Bauer, P.,
Poinar, H. N., \& Krause, J. (2011). A draft genome of
{\emph{Yersinia}}{ \emph{Pestis}} from victims of the {Black Death}.
\emph{Nature}, \emph{478}(7370), 506--510.
\url{https://doi.org/10.1038/nature10549}

\leavevmode\vadjust pre{\hypertarget{ref-bradley1721PlagueMarseillesConsider}{}}%
Bradley, R. (1721). \emph{The {Plague} at {Marseilles}: {Consider}'d
with {Remarks Upon} the {Plague} in {General}}. {W. Mears}.

\leavevmode\vadjust pre{\hypertarget{ref-brown2013AnthropologyInfectiousDisease}{}}%
Brown, P. J., \& Inhorn, M. C. (2013). \emph{The {Anthropology} of
{Infectious Disease}: {International Health Perspectives}}. {Routledge}.

\leavevmode\vadjust pre{\hypertarget{ref-hiderInPrepExaminingPathogenDNA}{}}%
Hider, J., Duggan, A. T., Klunk, J., Eaton, K., Long, G. S., Karpinski,
E., Golding, G. B., Prowse, T. L., Poinar, H. N., \& Fornaciari, G. (In
Prep). \emph{Examining pathogen {DNA} recovery across the remains of a
14th century {Italian} monk ({St}. {Brancorsini}) infected with
{Brucella} melitensis}.

\leavevmode\vadjust pre{\hypertarget{ref-murchieInPrepNoBonesIt}{}}%
Murchie, T., Karpinski, E., Eaton, K., Duggan, A. T., Baleka, S.,
Zazula, G., MacPhee, R. D. E., Froese, D., \& Poinar, H. N. (In Prep).
\emph{No bones about it: {Pleistocene} mitogenomes reconstructed from
the environmental {DNA} of permafrost}.

\leavevmode\vadjust pre{\hypertarget{ref-ober1982PlagueGranada13481349}{}}%
Ober, W. B., \& Aloush, N. (1982).
\href{https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1808550}{The plague
at {Granada}, 1348-1349: {Ibn Al-Khatib} and ideas of contagion.}
\emph{Bulletin of the New York Academy of Medicine}, \emph{58}(4),
418--424.

\leavevmode\vadjust pre{\hypertarget{ref-ortner2007DifferentialDiagnosisSkeletal}{}}%
Ortner, D. J. (2007). Differential {Diagnosis} of {Skeletal Lesions} in
{Infectious Disease}. In \emph{Advances in {Human Palaeopathology}} (pp.
189--214). {John Wiley \& Sons, Ltd}.
\url{https://doi.org/10.1002/9780470724187.ch10}

\leavevmode\vadjust pre{\hypertarget{ref-sampath2021PandemicsThroughoutHistory}{}}%
Sampath, S., Khedr, A., Qamar, S., Tekin, A., Singh, R., Green, R., \&
Kashyap, R. (2021). Pandemics {Throughout} the {History}. \emph{Cureus},
\emph{13}(9), e18136. \url{https://doi.org/10.7759/cureus.18136}

\leavevmode\vadjust pre{\hypertarget{ref-scott2001BiologyPlaguesEvidence}{}}%
Scott, S., \& Duncan, C. J. (2001). \emph{Biology of {Plagues}:
{Evidence} from {Historical Populations}}. {Cambridge University Press}.
\url{https://doi.org/10.1017/CBO9780511542527}

\leavevmode\vadjust pre{\hypertarget{ref-syed1981IslamicMedicine1000}{}}%
Syed, I. (1981). Islamic medicine: 1000 years ahead of its times.
\emph{Journal of the International Society for the History of Islamic
Medicine}, \emph{13}(1), 2--9.

\end{CSLReferences}

% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% Appendix
\begin{appendix}
\end{appendix}



\label{NumDocumentPages}

\end{document}
