<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Katherine Eaton">
  <meta name="dcterms.date" content="2021-05-15">
  <title>1000 Plagues in the Genomics Era</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reset.css">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/theme/black.css" id="theme">
  <link rel="stylesheet" href="pandoc/templates/revealjs-obsidian/simple.css"/>

</head>
<body>
  <div class="reveal">
    <div class="slides">

<section id="title-slide">
  <h1 class="title">1000 Plagues in the Genomics Era</h1>
  <p class="subtitle">Exhibiting Biographies of Bacteria</p>
  <p class="author">Katherine Eaton</p>
  <p>Curating Medieval Plague and Pestilence</p>

  <p class="date">15 May 2021</p>
</section>

<section>
<section id="introduction" class="title-slide slide level2">
<h2>Introduction</h2>
<aside class="notes" style="visibility: hidden">
Notes about the introduction section.
</aside>
</section>
<section id="what-is-ancient-dna" class="slide level3">
<h3>What is Ancient DNA?</h3>
<hr>
<ul>
<li>DNA retrieved from skeletal remains (+soil and artifacts!)</li>
<li>Genetic evidence offers a window into past disease experience.</li>
<li>Can be used to study infectious diseases, like medieval plague.</li>
</ul>
<p><img src="https://newhumanist.org.uk/images/rowson-DNA.jpg" width="40%" /></p>
<p><small><em>Illustration by Martin Rowson</em></small></p>
</section>
<section id="how-is-ancient-dna-applied" class="slide level3">
<h3>How is Ancient DNA applied?</h3>
<hr>
<ol type="1">
<li><strong>WHEN</strong> did plague first appear in human populations<span class="citation" data-cites="rasmussen2015EarlyDivergentStrains"><sup>1 </sup></span> ?</li>
<li><strong>WHERE</strong> has plague appeared<span class="citation" data-cites="bos2011DraftGenomeYersinia"><sup>2</sup></span> ?</li>
<li><strong>HOW</strong> has plague moved across the landscape<span class="citation" data-cites="guellil2020GenomicHistoricalSynthesis"><sup>3</sup></span> ?</li>
</ol>
<p><img src="https://github.com/ktmeaton/obsidian-public/raw/master/academic/guellil2020GenomicHistoricalSynthesis_fig2.jpg" width="40%" /></p>
<p><small><em>Guellil et al.Â (2020)</em></small></p>
</section>
<section id="what-evidence-is-needed" class="slide level3">
<h3>What evidence is needed?</h3>
<hr>
<p><span style="color:#00008B">Robust comparative datasets</span></p>
<ol type="1">
<li>Extensive sampling across time and place.</li>
<li>Careful curation of DNA sequences and metadata.</li>
</ol>
<blockquote>
<p>Fortunately, recent advances in DNA sequencing technology have facilitated efforts to conduct this extensive sampling on a Global scale, and data repositories continue to grow at unprecedented rates.</p>
</blockquote>
</section>
<section id="what-are-some-obstacles" class="slide level3">
<h3>What are some obstacles?</h3>
<hr>
<ul>
<li><p>This technological progress has created an avalanche of data.</p></li>
<li><p>Challenging to <u><strong>collect</strong></u>, <u><strong>curate</strong></u>, and <u><strong>present</strong></u> this information.</p></li>
<li><p>1600+ publicly available plague genomes.</p></li>
<li><p>Only 200-300 are routinely used in comparative frameworks.</p></li>
</ul>
</section>
<section id="what-can-we-do" class="slide level3">
<h3>What can we do?</h3>
<hr>
<table>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Step</th>
<th>Task</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1. Data Collection</td>
<td>Design Database software to assist geneticists with data collection.</td>
</tr>
<tr class="even">
<td>2. Curation</td>
<td>Curate historically-meaningful metadata for the available plague genomes.</td>
</tr>
<tr class="odd">
<td>3. Dissemination</td>
<td>Digital Exhibit</td>
</tr>
</tbody>
</table>
</section></section>
<section>
<section id="data-collection" class="title-slide slide level2">
<h2>Data Collection</h2>

</section>
<section id="what-is-a-dna-archive" class="slide level3">
<h3>What is a DNA archive?</h3>
<hr>
<ul>
<li>International Nucleotide Sequence Database Collaboration:
<ol type="1">
<li>USA - <a href="https://www.ncbi.nlm.nih.gov/">National Centre for Biotechnology Information</a></li>
<li>Europe - <a href="https://www.ebi.ac.uk/ena/">European Nucleotide Archive</a></li>
<li>Japan - <a href="https://www.ddbj.nig.ac.jp/dra/index-e.html">DNA Data Bank of Japan</a></li>
</ol></li>
</ul>
<p><img src="https://www.ddbj.nig.ac.jp/assets/images/center/insdc_shoukai.gif" width="40%"></p>
<aside class="notes" style="visibility: hidden">
Just as historians dive into the literary archives, geneticist tackle the DNA archives.
</aside>
</section>
<section id="how-is-the-archive-accessed" class="slide level3">
<h3>How is the archive accessed?</h3>
<hr>
<ul>
<li>Web-Browser experience is rich in informative metadata.</li>
<li>But there is little inter-record comparison or bulk record retrieval.</li>
</ul>
<p>!NCBI Search Bar.png</p>
<ul>
<li>In response, I designed database software.</li>
</ul>
<p><img src="https://github.com/ktmeaton/obsidian-public/raw/master/academic/NCBImeta%20Paper%20Header.png" style="width:60%;border: 1px solid #555"></p>
</section>
<section id="how-is-the-data-transformed" class="slide level3">
<h3>How is the data transformed?</h3>
<hr>
<p><img src="https://github.com/ktmeaton/obsidian-public/raw/master/academic/NCBImeta%20Data%20Conversion.png" style="width:80%;border: 1px solid #555"></p>
</section>
<section id="what-has-been-learned" class="slide level3">
<h3>What has been learned?</h3>
<hr>
<p><strong>The Good</strong></p>
<ul>
<li>Developed a strategic and reproducible data collection process.</li>
<li>Staying up to date with the latest and greatest <em>Y. pestis</em> research.</li>
</ul>
<p><strong>The Bad</strong></p>
<ul>
<li>Many research groups are re-inventing the wheel (myself included).</li>
<li>Search for ways to reduce redundancy, or if the uniqueness of each project and analysis precludes that.</li>
</ul>
<aside class="notes" style="visibility: hidden">
Strategic plan is very helpful for coordinating interdisciplinary projects.
</aside>
</section></section>
<section>
<section id="data-curation" class="title-slide slide level2">
<h2>Data Curation</h2>

</section>
<section id="process" class="slide level3">
<h3>Process</h3>
<hr>
<ol type="1">
<li><p>Cross-reference the automated database with primary sources.</p>
<ul>
<li>Correct discrepancies and gather additional context.</li>
</ul></li>
<li><p>Standardize metadata fields for analysis.</p>
<ul>
<li>Collection Location: <em>Peru Chota, San Juan de Licupis</em></li>
<li>Collection Date: <em>1999-04-10</em></li>
<li>Collection Host: <em>Neopsylla setosa</em></li>
</ul></li>
<li><p>Examine the DNA sequence data for quality.</p></li>
</ol>
</section>
<section id="collection-location" class="slide level3">
<h3>Collection Location</h3>
<hr>
<ol type="1">
<li><p>Select a <u><strong>geographic resolution</strong></u> that is appropriate for the study.</p>
<ul>
<li>Country, Province*, City, etc.</li>
</ul></li>
</ol>
<table>
<tbody>
<tr class="odd">
<td style="text-align: center;"><img src="https://raw.githubusercontent.com/ktmeaton/obsidian-public/master/academic/russia_country.jpg"></td>
<td style="text-align: center;"><img src="https://raw.githubusercontent.com/ktmeaton/obsidian-public/master/academic/russia_province.jpg"></td>
</tr>
<tr class="even">
<td style="text-align: center;"><small><em>Country (N=85)</em></small></td>
<td style="text-align: center;"><small><em>Province (N=85)</em></small></td>
</tr>
</tbody>
</table>
<ol start="2" type="1">
<li><p>Select a <strong>nomenclature</strong> system and coordinate reference.</p>
<ul>
<li>Google Maps, MapBox, OpenStreetMap*, etc.</li>
<li>Raw: <em>Peru Chota, San Juan de Licupis, Los Porongos</em></li>
<li>Standardized: <em>Peru, Cajamarca, Province of Chota, San Juan de Licupis</em></li>
<li>Coordinates: <em>-6.25, -78.833333</em></li>
</ul></li>
</ol>
</section>
<section id="collection-date" class="slide level3">
<h3>Collection Date</h3>
<hr>
<ol type="1">
<li><p>Select a <u><strong>temporal resolution</strong></u> that is appropriate for the study.</p>
<ul>
<li>Millennia, Century, Year*, Month, Day, etc.</li>
<li>Raw: <em>1999-04-10</em></li>
<li>Standardized: <em>1999</em></li>
</ul></li>
<li><p>Research how your analytical tools will handle <u><strong>uncertainty</strong></u>.</p>
<ul>
<li>Raw: <em>Between -2876 BCE and -2677 BCE.</em></li>
<li>Standardized: <em>-2776 BCE +/ 100</em></li>
</ul></li>
<li><p>Research how your analytical tools will handle years <u><strong>BCE</strong></u>.</p>
<ul>
<li>Raw: <em>-2776 BCE +/ 100</em></li>
<li>Standardized: <em>4797 YBP +/100</em></li>
</ul></li>
</ol>
</section>
<section id="collection-host" class="slide level3">
<h3>Collection Host</h3>
<hr>
<ol type="1">
<li><p>Select a <u><strong>biological resolution</strong></u> that is appropriate for the study.</p>
<ul>
<li>Family, Genus, Species etc.</li>
<li>Human vs.Â Non-Human*</li>
</ul></li>
<li><p>This is an extremely complex field because:</p>
<ul>
<li>Plague Ecology: Wild Rodents â Insects â Other Mammals</li>
<li>Taxonomic identification of a wide range of mammals is hard!</li>
<li>â<em>Meriones libycus</em>â vs.Â â<em>rat</em>â</li>
</ul></li>
</ol>
<p><img src="https://www.worldlifeexpectancy.com/images/a/w/b/meriones-libycus/meriones-libycus.jpg" width=30%></p>
</section>
<section id="dna-sequence-quality" class="slide level3">
<h3>DNA Sequence Quality</h3>
<hr>
<ol type="1">
<li><p>Select a <u><strong>genetic resolution</strong></u> that is appropriate for the study.</p>
<ul>
<li>Millennia, Century, Decade*, Year, Month, Day, etc.</li>
</ul></li>
<li><p>Process the DNA sequences using standardized workflows.</p></li>
</ol>
<p><img src="https://github.com/nf-core/eager/raw/master/docs/images/usage/eager2_workflow.png" width="50%"></p>
<ol start="3" type="1">
<li>Verify the results are congruent with the original publications.</li>
</ol>
</section>
<section id="what-has-been-learned-1" class="slide level3">
<h3>What has been learned?</h3>
<hr>
</section></section>
<section>
<section id="dissemination" class="title-slide slide level2">
<h2>Dissemination</h2>

</section>
<section id="digital-exhibit" class="slide level3">
<h3>Digital Exhibit</h3>
<hr>
</section></section>
<section id="conclusion" class="title-slide slide level2">
<h2>Conclusion</h2>

</section>

<section id="references" class="title-slide slide level2 unnumbered">
<h2 class="unnumbered">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-rasmussen2015EarlyDivergentStrains" class="csl-entry" role="doc-biblioentry">
1. Rasmussen, S., Allentoft, M. E., Nielsen, K., â¦ Willerslev, E. (2015). Early <span>Divergent Strains</span> of <span>Yersinia</span> pestis in <span>Eurasia</span> 5,000 <span>Years Ago</span>. <em>Cell</em>, <em>163</em>(3), 571â582. <a href="https://doi.org/10.1016/j.cell.2015.10.009">https://doi.org/10.1016/j.cell.2015.10.009</a>
</div>
<div id="ref-bos2011DraftGenomeYersinia" class="csl-entry" role="doc-biblioentry">
2. Bos, K. I., Schuenemann, V. J., Golding, G. B., â¦ Krause, J. (2011). A draft genome of <span><em>Yersinia</em></span><span> <em>Pestis</em></span> from victims of the <span>Black Death</span>. <em>Nature</em>, <em>478</em>(7370), 506â510. <a href="https://doi.org/10.1038/nature10549">https://doi.org/10.1038/nature10549</a>
</div>
<div id="ref-guellil2020GenomicHistoricalSynthesis" class="csl-entry" role="doc-biblioentry">
3. Guellil, M., Kersten, O., Namouchi, A., â¦ Bramanti, B. (2020). A genomic and historical synthesis of plague in 18th century <span>Eurasia</span>. <em>Proceedings of the National Academy of Sciences</em>, <em>117</em>(45), 28328â28335. <a href="https://doi.org/10.1073/pnas.2009677117">https://doi.org/10.1073/pnas.2009677117</a>
</div>
</div>
</section>
    </div>
  </div>

  <script src="https://unpkg.com/reveal.js@^4//dist/reveal.js"></script>

  // remote reveal.js plugins
  <script src="https://unpkg.com/reveal.js@^4//plugin/notes/notes.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/search/search.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/zoom/zoom.js"></script>
  // local reveal.js plugins  
  <script src="pandoc/plugins/reveal.js-menu/menu.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
      
        // Display a presentation progress bar
        progress: true,
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,

        // reveal.js plugins
        plugins: [
          RevealNotes,
          RevealSearch,
          RevealZoom,
          RevealMenu
        ]
      });
    </script>
    </body>
</html>
