---
title: "1000 Plagues in the Genomics Era: Exhibiting Biographies of Bacteria"
authors:
  - name: [[Katherine Eaton]]
    github: ktmeaton
    affiliations: "[[McMaster Ancient DNA Center]], [[McMaster University]]"
    github: ktmeaton
    orcid: 0000-0001-6862-7756
aliases:
  - Kalamazoo 2021 Abstract
tags:
  - üìù/üåø
status: idea
type: [[Abstract]]
lang: en-US
---

When used within a multi-disciplinary approach, genetic evidence provides an intriguing window into the [[disease experience]] of past populations. [[Ancient DNA]] techniques have been developed to identify [[bacterial]] and [[viral]] DNA in [[skeletal remains]], and even affected artefacts. These methods have additionally been used to explore the patterns of [[Spread|spread]] and [[Geographic Origin|geographic origins]] of several high-profile diseases, including [[Medieval|medieval]] [[Plague|plague]]. However, the ability to accurately reconstruct [[Geospatial|spatial patterns]] of an ancient disease requires [[robust comparative datasets]]. In order to produce such datasets, extensive sampling is required across both time and place, followed by careful [[Curation|curation]]. Fortunately, recent advances in DNA [[Sequencing|sequencing]] technology have facilitated efforts to conduct this extensive sampling on a [[Global]] scale, and [[data repositories]] continue to grow at unprecedented rates.

However, this technological progress has also placed a strain on our ability to effectively manage, [[curate]], and present the avalanche of data effectively. Despite there being over 1000 publicly available [[plague]] [[Genome|genomes]], only 200-300 are routinely used in comparative frameworks. We hypothesize this may be primarily attributed to the following three reasons, as the additional datasets are: 1) not known about, 2) known about, but difficult to access, or 3) accessible, but lack historically-meaningful [[Metadata|metadata]] . To tackle these obstacles, this project aims to do the following. First, to design [[Database]] [[Software|software]] to assist geneticists with the acquisition, organization, and discovery of [[Infectious Disease|infectious disease]] [[Metadata|metadata]]. Second, to [[Curate|curate]] historically-meaningful [[Metadata|metadata]] for the available plague genomes, both [[Ancient|ancient]]and [[Modern|modern]], and provide an accompanying [[Bibliography|bibliography]]. And third, to integrate these data into an updated [[Global|global]] [[Phylogenetic|phylogeny]], to broaden the scope of historical [[Narrative|narratives]] present.

To facilitate [[Curate|curation]] and to promote engagement with the project, two [[Digital Exhibit|digital exhibits]] are planned. The first exhibit is a [[Nextstrain]] [[Phylogenetic|phylogenetic]] presentation, which allows for an [[Interactive|interactive]] display of [[Disease|disease]] relationships between regions and over time. The second [[Digital Exhibit|digital exhibit]] is an [[ArcGIS]] [[Interactive|interactive]] [[StoryMap]], where [[Geospatial|spatial]] distribution can be interrogated along with an accompanying presentation of relevant historical records and [[Archaeology|archaeologically]] [[Artifact|artifacts]]. Overall, this project aims to contribute to our understanding of how and why [[Epidemic|epidemics]] of [[Plague|plague]] are related, with a particular focus on the [[Second Pandemic]]. Furthermore, by emphasizing the role that [[Digital Exhibit|digital exhibits]] play, we hope to open up new avenues for the integration of [[Multidisciplinary|multidisciplinary]] sources, [[Collaboration|collaborative]], and [[Pedagogy|pedagogical]] approaches to the study of [[Medieval|medieval]] [[Plague|plague]].
