<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="pandoc">
  <meta name="author" content="Katherine Eaton">
  <meta name="dcterms.date" content="2021-05-15">
  <title>1000 Plagues in the Genomics Era</title>
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reset.css">
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/reveal.css">
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <link rel="stylesheet" href="https://unpkg.com/reveal.js@^4//dist/theme/black.css" id="theme">
  <link rel="stylesheet" href="pandoc/templates/revealjs-obsidian/simple.css"/>

</head>
<body>
  <div class="reveal">
    <div class="slides">

<section id="title-slide">
  <h1 class="title">1000 Plagues in the Genomics Era</h1>
  <p class="subtitle">Exhibiting Biographies of Bacteria</p>
  <p class="author">Katherine Eaton</p>
  <p>Curating Medieval Plague and Pestilence</p>

  <p class="date">15 May 2021</p>
</section>

<section>
<section id="introduction" class="title-slide slide level2">
<h2>Introduction</h2>
<aside class="notes" style="visibility: hidden">
Notes about the introduction section.
</aside>
</section>
<section id="ancient-dna" class="slide level3">
<h3>Ancient DNA</h3>
<ul>
<li>Genetic evidence offers a window into past disease experience.</li>
<li>Ancient DNA (aDNA) techniques can identify bacterial and viral DNA in skeletal remains (and even artifacts!)</li>
<li>aDNA can then be used to study infectious diseases, like medieval plague.</li>
</ul>
<aside class="notes" style="visibility: hidden">
Notes about the Ancient DNA section
</aside>
</section>
<section id="applications" class="slide level3">
<h3>Applications</h3>
<ol type="1">
<li><strong>WHEN</strong> did plague first appear in human populations?<span class="citation" data-cites="rasmussen2015EarlyDivergentStrains"><sup>1 </sup></span></li>
<li><strong>WHERE</strong> has plague appeared?<span class="citation" data-cites="bos2011DraftGenomeYersinia"><sup>2</sup></span></li>
<li><strong>HOW</strong> has plague moved across the landscape?</li>
</ol>
</section>
<section id="requirements" class="slide level3">
<h3>Requirements</h3>
<ul>
<li>However, the ability to accurately reconstruct spatial patterns of an ancient disease requires robust comparative datasets.</li>
<li>In order to produce such datasets, extensive sampling is required across both time and place, followed by careful curation.</li>
<li>Fortunately, recent advances in DNA sequencing technology have facilitated efforts to conduct this extensive sampling on a Global scale, and data repositories continue to grow at unprecedented rates.</li>
</ul>
</section>
<section id="problem" class="slide level3">
<h3>Problem</h3>
<ul>
<li>However, this technological progress has also placed a strain on our ability to effectively manage, curate, and present the avalanche of data effectively.</li>
<li>Despite there being over 1000 publicly available plague genomes, only 200-300 are routinely used in comparative frameworks.</li>
</ul>
<p>We hypothesize this may be primarily attributed to the following three reasons, as the additional datasets are:</p>
<ol type="1">
<li>not known about</li>
<li>known about, but difficult to access</li>
<li>accessible, but lack historically-meaningful metadata .</li>
</ol>
</section>
<section id="objectives" class="slide level3">
<h3>Objectives</h3>
<p>To tackle these obstacles, this project aims to do the following.</p>
<ol type="1">
<li>First, to design Database software to assist geneticists with the acquisition, organization, and discovery of infectious disease metadata.</li>
<li>Second, to curate historically-meaningful metadata for the available plague genomes, both ancientand modern, and provide an accompanying bibliography.</li>
<li>And third, to integrate these data into an updated global phylogeny, to broaden the scope of historical narratives present.</li>
<li>Digital Exhibit</li>
</ol>
</section></section>
<section>
<section id="data-collection" class="title-slide slide level2">
<h2>Data Collection</h2>

</section>
<section id="dna-archives" class="slide level3">
<h3>DNA Archives</h3>
<ul>
<li>What and where are the DNA archives?</li>
<li>International Nucleotide Sequence Database Collaboration:
<ol type="1">
<li>USA - <a href="https://www.ncbi.nlm.nih.gov/">National Centre for Biotechnology Information</a></li>
<li>Europe - <a href="https://www.ebi.ac.uk/ena/">European Nucleotide Archive</a></li>
<li>Japan - <a href="https://www.ddbj.nig.ac.jp/dra/index-e.html">DNA Data Bank of Japan</a></li>
</ol></li>
</ul>
<p><img src="https://www.ddbj.nig.ac.jp/assets/images/center/insdc_shoukai.gif" width="40%"></p>
<aside class="notes" style="visibility: hidden">
Just as historians dive into the literary archives, geneticist tackle the DNA archives.
</aside>
</section>
<section id="archive-access" class="slide level3">
<h3>Archive Access</h3>
<ul>
<li>Power: Rich in informative metadata!</li>
<li>Problem: No inter-record comparison or bulk record retrieval.</li>
</ul>
<table>
<tbody>
<tr class="odd">
<td><img data-src="https://raw.githubusercontent.com/ktmeaton/NCBImeta/master/logo.png" alt="|300" /></td>
<td>!NCBImeta Paper Header.png</td>
</tr>
</tbody>
</table>
</section>
<section id="data-conversion" class="slide level3">
<h3>Data Conversion</h3>
<p>!NCBImeta Data Conversion.png</p>
</section></section>
<section>
<section id="data-curation" class="title-slide slide level2">
<h2>Data Curation</h2>

</section>
<section id="geography" class="slide level3">
<h3>Geography</h3>
<ul>
<li>How do you go from a written description</li>
</ul>
<pre><code>Bayan-Ölgii, Mongolia</code></pre>
<ul>
<li>to geographic coordinates?</li>
</ul>
<pre><code>48.5470083, 89.8549358</code></pre>
</section></section>
<section id="dissemination" class="title-slide slide level2">
<h2>Dissemination</h2>

</section>

<section id="conclusion" class="title-slide slide level2 unnumbered">
<h2 class="unnumbered">Conclusion</h2>
<div id="refs" class="references csl-bib-body hanging-indent" data-line-spacing="2" role="doc-bibliography">
<div id="ref-rasmussen2015EarlyDivergentStrains" class="csl-entry" role="doc-biblioentry">
1. Rasmussen, S., Allentoft, M. E., Nielsen, K., Orlando, L., Sikora, M., Sjögren, K.-G., Pedersen, A. G., Schubert, M., Van Dam, A., Kapel, C. M. O., Nielsen, H. B., Brunak, S., Avetisyan, P., Epimakhov, A., Khalyapin, M. V., Gnuni, A., Kriiska, A., Lasak, I., Metspalu, M., … Willerslev, E. (2015). Early <span>Divergent Strains</span> of <span>Yersinia</span> pestis in <span>Eurasia</span> 5,000 <span>Years Ago</span>. <em>Cell</em>, <em>163</em>(3), 571–582. <a href="https://doi.org/10.1016/j.cell.2015.10.009">https://doi.org/10.1016/j.cell.2015.10.009</a>
</div>
<div id="ref-bos2011DraftGenomeYersinia" class="csl-entry" role="doc-biblioentry">
2. Bos, K. I., Schuenemann, V. J., Golding, G. B., Burbano, H. A., Waglechner, N., Coombes, B. K., McPhee, J. B., DeWitte, S. N., Meyer, M., Schmedes, S., Wood, J., Earn, D. J. D., Herring, D. A., Bauer, P., Poinar, H. N., &amp; Krause, J. (2011). A draft genome of <span><em>Yersinia</em></span><span> <em>Pestis</em></span> from victims of the <span>Black Death</span>. <em>Nature</em>, <em>478</em>(7370), 506–510. <a href="https://doi.org/10.1038/nature10549">https://doi.org/10.1038/nature10549</a>
</div>
</div>
</section>
    </div>
  </div>

  <script src="https://unpkg.com/reveal.js@^4//dist/reveal.js"></script>

  // remote reveal.js plugins
  <script src="https://unpkg.com/reveal.js@^4//plugin/notes/notes.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/search/search.js"></script>
  <script src="https://unpkg.com/reveal.js@^4//plugin/zoom/zoom.js"></script>
  // local reveal.js plugins  
  <script src="pandoc/plugins/reveal.js-menu/menu.js"></script>

  <script>

      // Full list of configuration options available at:
      // https://revealjs.com/config/
      Reveal.initialize({
      
        // Display a presentation progress bar
        progress: true,
        // Display the page number of the current slide
        slideNumber: true,
        // Push each slide change to the browser history
        history: true,

        // reveal.js plugins
        plugins: [
          RevealNotes,
          RevealSearch,
          RevealZoom,
          RevealMenu
        ]
      });
    </script>
    </body>
</html>
