# NML Inventory

Title: National Microbiology Laboratory Omics, Bioinformatics, and Computational Biologist Inventory
Link: <https://emploisfp-psjobs.cfp-psc.gc.ca/psrs-srfp/applicant/page1800?poster=1567079>

| Advertisement Name | Reference Number | Organization | 
| ------------------ | ---------------- | ------------ |
National Microbiology Laboratory Omics, Bioinformatics, and Computational Biologist Inventory	| AHS21J-019669-000027	| 	Public Health Agency of Canada 	|

## Description

>"The National Microbiology Laboratory is committed to providing exciting opportunities in an open and collaborative environment. You’ll be at the forefront of scientific computing for use in investigating the world’s most important infectious diseases.  
  
> We are seeking highly motivated individuals to lead the application and development of genomic and bioinformatics tools, algorithms, and analytical methods to support the Agency’s programs for public health research, surveillance, diagnostics and outbreak response.  
  
> The incumbent will support science leads in collaboratively designing, developing and conducting computational data analytics for large data sets in the “omics” fields (genomics, transcriptomics, proteomics), provide expert advice, drive technology adoption in science programs, and manage projects and/or supervise multi-disciplinary teams.  
  
> To build national capacity to address the COVID-19 pandemic, we are immediately seeking individuals for bioinformatics analysis of complex genomics data. This important work will help to identify and track transmission trends and inform critical decision making relevant to health authorities across Canada during the pandemic.  
  
>To see examples of some of our software please visit: [phac-nml](https://github.com/phac-nml)"

## Resume

I typed up a plain-text [[Resume]].

## Screening Questions

> **Recent and significant experience working in the area of comparative “omics” (genomics, proteomics, transcriptomics, metagenomics, metatranscriptomics, etc.), bioinformatics, computational biology or population genetics.**
> I am currently completing my PhD in the field of ancient pathogen genomics at the McMaster Ancient DNA Centre. This unique institute has given me the opportunity to learn specialized laboratory techniques for next-generation sequencing of ancient DNA from fossil specimens. I have also been encouraged to develop novel computational techniques, which has led me to develop and publish database software. I am currently finishing my final dissertation manuscript on the reconstruction of ancient bacterial genomes and the pipeline I created to analyze this large and complex dataset.

>Recent and significant* experience with bioinformatics methods and databases, and using open-source bioinformatics software (installation and usage).
 My PhD involves analyzing neglected datasets found in the NCBI databases such as the SRA. I frequently access these resources both in the browse, and with command-line APIs. Furthermore, I designed novel database software to access and organize these databases for high-throughput projects.
 The software I interact with most are within the fields of genome alignment (ex. bwa), metagenomics (ex. kraken), and phylogenetic analysis (ex. IQTREE). I am an advocate for containerizing applications with conda, docker, or singularity although I am capable of compiling from source when needed. Because of my passion for bioinformatics software, I often assist other co-workers in acquiring, installing, and testing software for their own projects.

-	Developed the tool 'NCBImeta', for efficient and comprehensive metadata retrieval from NCBI databases. Published in an open-access and peer-reviewed journal.
-	Developed a snakemake pipeline to process SRA data into phylogenies.
-	Experience containerizing software and dependencies using conda, docker, and singularity.

>Recent and significant\* experience in the assembly, annotation, analysis and interpretation of complex “omics” datasets such as pathogen and/or host genomes.
> For one of my PhD dissertation manuscripts,  I am assembling and aligning 600+ bacterial genomes for global phylogenetic reconstruction. This work involved genome annotation, such as identifying key variants related to virulence, and geospatial analysis of pathogen spread between countries.

>**Recent and significant* experience in applying project management practices to data acquisition and analysis.**
> I am part of a large multidisciplinary team that is analyzing the history of infectious disease in Denmark. Through working with this team,  two practices emerged as being the most critical. The first practice was early and frequent communication. We found a 'sweet spot' of weekly meetings for small project teams, and monthly check-ins with all collaborators. The second practice was consolidating and sharing data. Organizing results into a central place has decreased experiment redundancy and generated many new ideas.

> **Recent and significant* experience in assessing “omics” data quality, downstream data processing, and developing or maintaining data quality standards.**
> Throughout my PhD, I generated and evaluated Illumina sequencing data for over 1000 highly-degraded fossil specimens. A critical part of this workflow was assessing DNA quality before sequencing (PCR) in addition to NGS metrics such as base quality, alignment and assembly quality, and genotype quality in SNP-typing applications.

>**Recent and significant experience in planning, organizing, and designing scientific or software projects.**
> I designed multiple laboratory-based projects in my PhD that concerned pathogen detection in ancient biological specimens. These research projects include PCR optimization and screening, shotgun metagenomics evaluation, and targeted enrichment for genome capture.
>To expand on the laboratory work, I developed database tools and pipelines to identify publicly available datasets and perform phylogenetic analyses. Projects that include non-sensitive data are published open-source on Github, with supporting documentation, issue trackers, and changelogs.

>Recent and significant experience in coordinating, and supervising or managing multidisciplinary teams or projects.
**Yes**
At the present moment, I am actively engaged with three multidisciplinary teams and I fill three general roles. The first is coordination, which includes scheduling meetings, creating agendas, and developing action plans. The second is communication, where I help develop publication plans and draft manuscripts. The third is research, where I assist in identifying datasets and designing experiments relevant to the shared publication goals.

>Graduation with a degree from a recognized post-secondary institution with specialization in a biological science (ex. Microbiology, Virology, Parasitology, Biochemistry, Immunology, Genetics/Genomics, Molecular Biology, Bioinformatics, or Biology or other related discipline); or
**Yes
I am currently completing my PhD in the McMaster Ancient DNA Centre, at McMaster University. My field of research is genomic sequencing of ancient pathogenic bacteria. My secondary focus is the development of bioinformatic tools and pipelines to streamline the process of data acquisition all the way through to phylogenetic analyses.
I will have filed for defense on June 25, with an anticipated defense in August, and final submission in September.**

> For multidisciplinary positions classified as BI, the minimum standard is:
Graduation with a degree from a recognized post-secondary institution in a natural, physical or applied science with specialization in a field relevant to the duties of the position (ex. Computer Science, Bioinformatics, Computational Biology, Biomedical Engineering, Biotechnology).
**Yes**

---
### Location
> 1. Are you willing to consider positions that would require temporary mobilization to fulfill operational requirements including outbreak and emergency laboratory response?
**No**

> 2. Are you willing to consider positions that would require permanent relocation to fulfill operational requirements?
**Yes**

> 3. Are you currently employed within the Public Health Agency of Canada?
**No**

> 4. Are you currently employed within the Federal Public Service?
> **No**

> 5. Do you have a current Federal Government Security Clearance?
**No**

> 6. Have you been found qualified in any pools within the Public Health Agency of Canada?
**No**

> 7. Have you been found qualified in any pools within the Public Health Agency of Canada?
**No**

> 8. Have you ever been a student with the federal government?
**No**

> 9. Are you interested in casual employment (90 working day opportunity)?
**Yes**

> 10. Have you ever been a student with the federal government?
**No**

> 11. English Essential positions?
**Yes**

> 12. French essential positions?
**No**

> 13. Bilingual (English and French) positions?
**No**

> 14. Vancouver, British Columbia?
**No**

> 15. British Columbia - Other locations?
**No**

> 16. Edmonton, Alberta?
**Yes**

> 17. Calgary, Alberta?
**Yes**

> 18. Lethbridge, Alberta?
**No**

> 19. Alberta - other locations?
**No**

> 20. Regina, Saskatchewan?
**No**

> 21. Saskatchewan - other locations?
**No**

> 22. Winnipeg, Manitoba?
**Yes**
> 23. Manitoba - other locations?
**No**

> 24. Guelph, Ontario?
**No**

> 25. Greater Toronto Area, Ontario?
**No**

> 26. Ottawa (Ottawa-Gatineau), Ontario?
**Yes**

> 27. Ontario - other locations?
**No**

> 28. Saint-Hyacinthe, Québec?
**No**

> 29. Montreal (or surrounding locations), Quebec?
**Yes**

> 30. Quebec - other locations?
**No**

> 31. Fredericton, New Brunswick?
**No**

> 32. Moncton, New Brunswick?
**Yes**

> 33. New Brunswick - other locations?
**No**

> 34. Halifax, Nova Scotia?
**Yes**

> 35. Nova Scotia - other locations?
**Yes**

> 36. Charlottetown, PEI?
**Yes**

> 37. P.E.I. - other locations?
**No**

> 38. St John's, Newfoundland?
**Yes**

> 39. Newfoundland - other locations?
**No**

> 40. Territories?
**No**

> 41. Graduation with a MSc or PhD from a recognized post-secondary institution with a specialization in a field relevant to the duties of the position(s).
**Yes
I am currently completing my PhD in the McMaster Ancient DNA Centre, at McMaster University. My field of research is genomic sequencing of ancient pathogenic bacteria. My secondary focus is the development of bioinformatic tools and pipelines to streamline the process of data acquisition all the way through to phylogenetic analyses.
I will have filed for defense on June 25, with an anticipated defense in August, and final submission in September.**

---

### Bioinformatics, Dev, Database

>1. Experience using Galaxy Project (https://galaxyproject.org/), and /or IRIDA (https://www.irida.ca/) or other open-source bioinformatics software.
**Yes**

>2. Experience using UNIX or LINUX-based operating systems and commands.
**Yes**

>3. Experience programming in one or more of the following languages: Perl, Python, R, Bash, C, C++, Java, HTML, JavaScript, Structured Query Language (SQL), or other relevant scientific programming environments.
**Yes**

>4. Experience managing relational databases such as MySQL, MariaDB, PostgreSQL, Oracle, etc.
**Yes**

>5. Experience in the development of large-scale genomic databases. 
**Yes**

>6. Experience in developing algorithms and software tools for the analysis of pathogen and/or host genomes.
**Yes**

>7. Experience with high performance computing.
**Yes**

>8. Experience managing or maintaining database software in a biological laboratory setting.
**Yes**

>9. Experience working in a professional software development environment.
**Yes**

>10. Experience working with bioinformatics tools to manipulate, clean, organize, evaluate and visualize data. Do you meet this requirement?
**Yes**

>11. Experience in designing and implementing relational database schemas and writing Structured Query Language (SQL) queries.
**Yes**

>12. Experience in developing internet-based client: server applications and programming interfaces for the analysis of “omic” information. 
**Yes**

>13. Experience designing and developing public health informatics applications to collect, analyze and model public health data.
**NO**

>14. Experience with user interface or business logic development for public health surveillance and reporting applications.
**No**

>15. Experience with mobile development.
**No**

>16. Experience working within an omics-based core-facility or lab environment.
**Yes**

---
### Omics
>17. Experience in preparing biological specimens for whole genome sequencing. Do you meet this requirement?
**Yes**

>18. Experience with performing Next Generation Sequencing. Do you meet this requirement?
 **Yes**
 
>19. Experience managing and curating large datasets. Do you meet this requirement?
**Yes**

>20. Experience in analysing and interpreting whole genome sequence data. Do you meet this requirement?
**Yes**

>21. Experience analyzing and interpreting next generation sequence variant data including SNP imputation, GWAS (genome-wide association studies) and genomic prediction for complex traits. Do you meet this requirement?
**Yes**

>22. Experience in performing and interpreting phylogenetic and phylogenomic analysis. Do you meet this requirement?
**Yes**

>23. Experience in in analysing and interpreting molecular and genomic subtyping results. Do you meet this requirement?
**Yes**

>24. Experience in epigenetics. Do you meet this requirement?
**No**

>25. Experience in proteomics or metabolomics data analysis and visualization. Do you meet this requirement?
**No**

---
### Program-Specific
>26. Experience in developing bioinformatics in support of pathogen surveillance and antimicrobial resistance prediction. Do you meet this requirement?
**Yes**

>27. Experience in the analysis of microbial (bacterial, viral, and/or fungal) genomes. Do you meet this requirement?
**Yes**

>28. Experience in performing and interpreting phylogenetic and phylogenomic analysis of microbial (bacterial, viral, and/or fungal) pathogens. Do you meet this requirement?
**Yes**

>29. Experience in developing algorithms and software tools for the analysis of bacterial genomes. Do you meet this requirement?
**Yes**

>30. Experience in the characterization of bacterial pathogens. Do you meet this requirement?
**Yes**

>31. Experience with scientific research in the area of human, foodborne enteric pathogens. Do you meet this requirement?
**No**

>32. Experience developing and implementing in silico methods for molecular typing and characterization of food- and water-borne enteric pathogens. Do you meet this requirement?
**No**

>33. Experience in the development and optimization of molecular diagnostics assays. Do you meet this requirement?
**Yes**

---
### Project Coordination and Communication

>34. Experience in communicating and consulting with Canadian public health laboratories, federal/provincial /territorial or international partners, or research laboratories for the purposes of knowledge transfer and exchange or collaboration. Do you meet this requirement?
**Yes**

>35. Experience as a project manager of national or international lab activities. Do you meet this requirement?
**No**

>36. Experience coordinating field diagnostics or mobile laboratory activities. Do you meet this requirement?
**No**

>37. Experience in coordinating science program(s) or function(s) such as research projects or research project logistics or program logistics. Do you meet this requirement?
**Yes**

>38. Experience in providing specialized scientific, technical expertise and/or recommendations. Do you meet this requirement?
**Yes**

>39. Experience in the communication of scientific findings and program information to clients, senior management. Do you meet this requirement?
**Yes**

>40. Experience writing and being published in peer-reviewed scientific journals and delivering presentations at scientific meetings. Do you meet this requirement?
**Yes**

>41. Experience in assessing, analyzing and contributing to the development of policy or programs relating to science or public health. Do you meet this requirement?
**No**

>42. Experience participating and assisting in working groups, committees or project teams led by the principal investigator, management or stakeholders. Do you meet this requirement?
**Yes**

---
### Laboratory
>43. Experience in providing support for laboratory-related outbreak preparedness and response activities. Do you meet this requirement?
**No**

>44. Experience in planning, coordinating, and conducting scientific or research projects in a laboratory setting. Do you meet this requirement?
**Yes**

>45. Experience in performing analysis methods or experiments to develop, modify or validate new laboratory or computational approaches, methodologies, directions, techniques or procedures. Do you meet this requirement?
**Yes**

>46. Experience managing or maintaining database software in a biological laboratory setting. Do you meet this requirement?
**Yes**

>47. Experience maintaining instrumentation and monitoring instrument performance. Do you meet this requirement?
**Yes**

>48. Experience managing and supervising a group performing lab activities. Do you meet this requirement?
**Yes**

>49. Experience in providing guidance and training to scientific personnel. Do you meet this requirement?
**Yes**

---
### Program Operations
>50. Experience in financial management (ex. diagnostic testing budgets, research project/outbreak response expenses, program budgets). Do you meet this requirement?
**Yes**

>51.  Experience managing human, financial and capital resources. Do you meet this requirement?
**No**

---
### Hours
>1. Must be available during core hours. Do you meet this requirement?
**Yes**

>2. Willingness and ability to occasionally work overtime. Do you meet this requirement?
**Yes**

>3. Willingness and ability to assist, mobilize, or deploy to fulfill operational requirements including outbreak and emergency laboratory responses. Do you meet this requirement?
**No**

---

tags:
  - [[Job]]