---
title: "PhD Research Proposal"
authors: 
  - name: Katherine Eaton
    github: ktmeaton
    orcid: 0000-0001-6862-7756
    affiliations: [[[McMaster Ancient DNA Center]], [[McMaster University]], [[Department of Anthropology]], [[McMaster University]]]
aliases:
  - Pathogen Detection in Paleomicrobiology
  - Comprehensive Exam 2
tags:
  - üìù/üå±
  - ‚¨ú/üß® 
due: 2021-05
status: priority
type: [[Paper]]
lang: en-US
repo: ktmeaton/obsidian-public
filepath: academic/[[PhD Research Proposal]]
toc: true
numberSections: true
sectionsDepth: 3
cmd: "pandoc/manubot.sh 'PhD Research Proposal.md' library.json ../../rootstock"
---

## Abstract

Pandemics of plague have emerged at least three times throughout human history with tremendous mortality and extensive geographic spread. Despite more than a century of comprehensive research, the origins and spatiotemporal patterning of these pandemics remain obscure. The First Pandemic (541 - 750 CE) devastated the Byzantine Empire of the Mediterranean world, the Second Pandemic (14 - 18th century) swept across much of Afro-Eurasia, and the Third Pandemic (19thcentury - present) has reached every continent except for Antarctica, and continues to persist in various endemic foci around the world. Here I propose a series of three phylogeographic projects aimed at investigating the epidemiology of plague within and between these three pandemics, emphasizing the origins, patterns of dispersal, and genetic continuity over time. Using a collection of bacterial isolates, as well as archaeological remains of suspected victims, this project aims to retrieve and sequence the genomes of the causative agent of plague, Yersinia pestis, from under-represented, but highly informative, plague epidemics in the past. The results of this research will expand our epidemiological knowledge of plague, in a way that improves our understanding of the interplay of factors contributing to modern re-emergences, as well as the historical events that triggered past pandemics. This project seeks not simply to reconstruct how epidemics are interrelated, but why these relationships exist using the frameworks of ecosocial epidemiology and the anthropology of infectious disease. The combination of molecular epidemiology methods driven by anthropologically-focused questions offers a unique lens through which to reconstruct the fluctuating patterns of human connectivity and ecological interaction that have shaped our relationship with infectious disease.

## Theoretical Overview

The overarching narrative that unites the following three projects is the exploration of socioecological factors driving the epidemiology of plague. Thus far, the field of plague phylogenetics has been firmly entrenched in descriptive epidemiology; primarily focused on what relationships exist and how they have occurred through an evolutionary framework. While notable exceptions do exist (Morelli et al., 2010), the absence of any explicitly-defined theoretical framework has resulted in a body of literature devoid of social processes, or ones that are mentioned in passing at best. This comes as a surprise, given the growing recognition of socio-ecological context in the fields of emerging infectious diseases (Parkes et al., 2005), archival plague research (Campbell, 2016), and modern plague epidemiology (McCauley et al., 2015). Previously, interpretive plague genomics was hindered by a lack of available data, but the explosion of genomes (300+) produced via recent technological developments no longer bears the same burden. Plague phylogenetics has now reached a critical point, in which voluminous data is publicly accessible, and new theoretical frameworks are being actively developed in adjacent fields.

In response, the objective of this research project is not solely to characterize how plague epidemics are interrelated, but why these relationships exist. This dissertation aims to expand beyond traditional epidemiological theory in order to intepret the genomic history of an infectious pathogen using ecosocial epidemiology (Krieger, 1994; Krieger, 2001) and the anthropology of infectious disease (Brown and Inhorn, 2013). While seemingly originating from disparate fields, these frameworks highlight the importance of explaining patterns of disease distribution by examining changing patterns of interactions in the environment, both social and natural. I will consider political economic processes known to have played an influential role in disease dissemination such as human connectivity via trade routes, as well as the fluctuating influence of imperialism (ex. Ottoman Empire, European Colonial Empire). Attention will also be paid to changing environmental interactions such as the exploitation of natural resources (Mitchell, 2002) and the interacting effects of climate and intensification of agricultural land use (McCauley et al., 2015; Young et al., 2017). These environmental effects have quickly risen to the forefront of discussions of plague epidemiology in high-incidence areas, such as eastern Africa, and promote novel hypotheses generated from a robust understanding of plague ecology. Overall, through the incorporation of an ecosocial framework, I am to challenge the assumptions underlying current epidemiological models invoked in plague genomics, and to propose new testable hypotheses aimed at slowly uncovering the hidden processes underlying an immensely complex disease.

## Project 1 - Unraveling the Narrative of the Modern Pandemic

### Introduction

It has recently been argued that the modern pandemic of plague is unambiguously and exclusively caused by strains of Y. pestis dispersed out of Hong Kong in 1894 (Achtman, 2016). This statement appears to hold true for epidemics with socio-historical links to Hong Kong including Madagascar, India, Germany, Netherlands, North America, and South America (Morelli et al., 2010). However, this conclusion is primarily drawn from outbreaks occurring in the early to mid-twentieth century. As a result, the origins of the recent re-emergent epidemics, for example the pneumonic outbreak of plague in India in the 1990s (Ramalingawami, 1995), and their interrelationships remain unclear. But more importantly, this presumption neglects the experience of Central and East Africa, as well as Western Asia, where strains of plague appear to be more ancient in origin. Intriguingly, plague in these areas may have diverged at similar times in the past, but potentially as separate evolutionary events from different parent populations (Cui et al., 2013). To add to this complexity, the presence of heterogeneous lineages of Y. pestis in Turkey and the Caucuses suggest multiple, distinct reintroductions to the area. The assumption of a homogenous global epidemiology of plague therefore downplays the significance of these underrepresented, but informative, regions which demonstrate that the distribution of modern plague was influenced by multiple disease dispersal events in history.

Overall, this strong geographic and temporal sampling bias has left numerous questions outstanding concerning where plague foci exist outside of Central Asia, how long they have existed, and whether they have seeded modern plague epidemics, and possibly past pandemics. In response, this research project aims to phylogenetically characterize the evolutionary relationships of Y. pestis isolates of undersampled geographic regions and unexamined recent re-emergences that have occurred during the Modern Pandemic of the 20th century.

...

## Project 2 - Disease Dispersal During the European Second Pandemic

### Introduction 

Overall, there is tremendous variation in the proposed routes of the Second Pandemic due to the sparseness and ambiguity of surviving mortality records (Christakos et al., 2005). It was long hypothesized that subsequent waves of plague that ravaged Europe were the result of reintroductions from across the Mediterranean or in Central Asia and was built on the modern assumption that there was no plague reservoir in Europe. But this hypothesis has recently been challenged by evidence for potential plague continuity within Europe (Bos et al., 2016) and new analyses aimed at identifying the locations of these putative reservoirs (Carmichael, 2014). While this research marks a considerable theoretical shift, it is still unknown to what extent the distribution of the Second Pandemic can be attributed to local persistence or recurrent long-distance introductions.

### Context

In October 1347, merchant vessels brought plague to Italy after fleeing the Siege of Kaffa in the Crimea (Derbes, 1966). From this entry point, the disease spread relentlessly throughout Europe‚Äôs population with mortality estimates ranging from 30-50% (Benedictow, 2004). Within 5 years, plague had spread outward from southern Italy to reach every corner of Europe. The disease continued to reemerge as local epidemics with varying severity and finally retreated from Europe by the end of the 18th century. However, a controversial issue has recently arisen over these patterns of plague spread. Iconic maps produced in the mid-20th century have largely influenced the discussion of geographic and temporal spread of plague (Mengel, 2011). But the sources used in the construction of these maps have recently undergone critical reexamination (Christakos et al., 2005, Skog and Hauska, 2013, Bossak and Welford, 2015). These sources predominantly consist of contemporaneous archival documents such as parish records, financial transactions, wills, and local testimonies (Christakos et al., 2005). Unfortunately, these documents are often sparse and of poor quality leading to high levels of uncertainty (Skog and Hauska, 2013). In addition, the majority of these sources are non-specific with regards to the cause of death. Therefore, there is a need for reliable evidence that can be firmly associated with plague, and ancient DNA is an ideal candidate.

Ancient DNA analysis has strongly implicated the bacterium Y. pestis as the causative agent of the Second Pandemic in Europe (Haensch et al., 2010; Schuenemann et al., 2011). Single nucleotide polymorphisms present in the plague bacterium have been used to link related strains and infer distinct waves of infection (Haensch et al., 2010; Morelli et al., 2010). Furthermore, advances in DNA high-throughput sequencing technology now enable the retrieval of entire genomes of ancient plague (Bos et al., 2011; Wagner et al., 2014). This advancement has greatly improved our ability to discover these novel genetic mutations which will help to explain the spatiotemporal distribution of historical pathogens. In addition, modern epidemiological studies have shown great promise in using genomic data to track routes of infection (Cui et al., 2013; Alfonso-Morales et al., 2013; Lewis et al., 2015; Lowell et al., 2015). Therefore, this research project aims to apply this methodology in producing robust genomes of ancient Y. pestis to reconstruct the spread of the Second Pandemic.