---
project: [[Plague Denmark]]
due: June 28, 2021
authors:
  - name: [[Katherine Eaton]]*
    orcid: 0000-0001-6862-7756
    affiliations:
      [	
        "[[McMaster Ancient DNA Center]]",
        "[[Department of Anthropology]], [[McMaster University]]"
      ]
  - name: [[Ravneet Sidhu]]
    affiliations:
    affiliations:
      [	
        "[[McMaster Ancient DNA Center]]",
        "[[Department of Biology]], [[McMaster University]]"
      ]
  - name: [[Jennifer Klunk]]
    orcid: 0000-0002-6521-8516
    affiliations: [
      "[[Daicel Arbor Biosciences]]"
    ]
  - name: [[Julia Gamble]]
    orcid: 0000-0001-7486-757X
    affiliations: [
      "[[University of Manitoba]]"
    ]
  - name: [[Jesper Boldsen]]
    orcid: 0000-0002-2850-0934
    affiliations: [
      "[[University of Southern Denmark]]; [[ADBOU]]"
    ]
  - name: [[Ann Carmichael]]
    affiliations: [ "Department of History, [[Indiana University Bloomington]]" ]
  - name: [[NÃ¼khet VarlÄ±k]]
    orcid: 0000-0001-6870-5945
    affiliations: [ "Department of History, [[Rutgers University-Newark ]]" ]
  - name: [[Sebastian Duchene]]
    orcid: 0000-0002-2863-0907
    affiliations:
      [
        "[[The Peter Doherty Institute for Infection and Immunity]],
          [[University of Melbourne]]"
      ]
  - name: [[Leo Featherstone]]
    orcid: 0000-0002-8878-1758
    affiliations:
      [
        "[[The Peter Doherty Institute for Infection and Immunity]],
          [[University of Melbourne]]"
      ]
  - name: Vaughan Grimes	
    orcid: 0000-0002-2177-3147
    affiliations: [
      "[[Memorial University]]"
    ]
  - name: [[Brian Golding]]
    orcid: 0000-0002-7575-0282
    affiliations:
      [
        "[[Department of Biology]], [[McMaster University]]"
      ]
  - name: [[Sharon DeWitte]]
    orcid: 0000-0003-0754-8485	
    affiliations: [
        [[University of South Carolina]]
    ]
  - name: [[Hendrik Poinar]]
    orcid: 0000-0002-0314-4160
    affiliations:
      [
        "[[McMaster Ancient DNA Center]]",
        "[[Department of Anthropology]], [[McMaster University]]"
      ]
lang: en-US
repo: ktmeaton/obsidian-public
filepath: academic/[[Plague Denmark Paper]]
tags: â¬œ/ðŸ§¨ 
status: priority
title: The Plague in Denmark (1000-1800)
subtitle:  _Yersinia pestis_ something
type: [[Task]]
compile-manubot: "conda activate manubot && pandoc/manubot.sh 'Plague Denmark Paper Outline.md' pandoc/bib/library.json ../../rootstock"
url: https://ktmeaton.github.io/obsidian-public/academic/Plague%20Denmark%20Paper.html
---

## Introduction

### Second Pandemic

- The Second Pandemic of plague devastated Europe (14th - 18th centuries)

### Denmark

- The etiology and epidemiology of plague in Scandinavia is contentious.
- Historical sources were sparse and unreliable [@christensen2003ThesePerilousTimes].
- Outbreaks the 14<sup>th</sup> and 15<sup>th</sup> centuries were poorly documented.
- "disease terminology is certainly vague and ambiguous... pestis, pestilence and derivatives... were also used to characterize a number of diseases"

- Textual evidence became much more plentiful and precise in the 16<sup>th</sup> century. At this point, the origins and frequency of epidemics could be more clearly identified. 
- *"The outbreaks clearly coincided with larger European incidences of the disease and often the sources expressly state that the infection had come to Denmark from the Baltic region or northern Germany."* [@slavin2021OutWestFormation]
- Plague outbreaks re-occurred approximately every 10 years, lasting 1-5 years  [@christensen2003ThesePerilousTimes; @lenz2016BlackDeath]

- "The plague history of Sweden and Denmark is much less studied than Norwayâ€™s, but the incidence of importation, seasonality, and temporal rhythms of plague epidemics were clearly different, especially after 1400.47 The reason was evidently that these countries did not have such strong commercial connections with England and mostly imported plague from the Baltic region and the Hanseatic cities, and, in the later part of the plague pandemic, also from the Netherlands." [@benedictow2016BlackDeathLater]



- One of the most famous digitized databases has no data from Denmark [@buntgen2012DigitizingHistoricalPlague; roosen2018DangersNoncriticalUse].

### Objectives

- We tested for the presence of _Y. pestis_ in medieval and early modern Denmark.
- We used molecular assays to screen the skeletal remains of 298 individuals across 13 archaeological sites, dated to between 1000 and 1800 century CE.
- 

## Results and Discussion {.page_break_before}

We detected _Y. pestis_ in 7 archaeological sites using PCR assays and targeted sequencing (Figure 1A). Across the 7 sites, 8.3% of individuals (13/157) tested positive for _Y. pestis_, ranging from 3.3% at Ribe LindegÃ¤rden to 14.3% at HÃ¥gerup. Of these plague-positive individuals, 9 had sufficient sequencing depth (>3X) of the _Y. pestis_ chromosome for phylogenetic analysis (Figure 2B). To estimate a time-scaled phylogeny and dates for each sample, we fit a relaxed molecular clock to an alignment of 49 _Y. pestis_ genomes dated to the Second Pandemic (Figure 1B).

We found no evidence of _Y. pestis_ in Denmark between 1000 and 1300 CE. However, the factors influencing the preservation of ancient DNA are wide-ranging and complex [@dabney2013AncientDNADamage] and the absence of evidence cannot prove evidence of absence. That being said, we sampled a minimum of 85 individuals and a maximum of 165 individuals that pre-date the 14<sup>th</sup> century (Figure 2A). Taking the mean positivity rate observed in this study (8.3%), we would expect to detect _Y. pestis_ in 7 to 13 individuals from this time frame if it were present. We therefore interpret this as tentative evidence that _Y. pestis_ was a relatively new pathogen in medieval Denmark, that did not become abundant and/or widespread until at least the 14<sup>th</sup> century.

The earliest evidence of _Y. pestis_ in Denmark was found in the town of Ribe. Two individuals were associated with _Y. pestis_ from the first half of the 14<sup>th</sup> century, dated to 1333 (1301-1366) and 1350 (1319-1383). These estimates are highly congruent with the historical record, as the first documented appearance of plague in Denmark was at Ribe in 1349. Furthermore, these strains fell within an unresolved cluster (posterior: 0.15) of samples from Northern and Western Europe (Figure 3) which has previously been linked to the Black Death (1343-1356) [@spyrou2019PhylogeographySecondPlague]. Our molecular dates support this historical association, albeit only weakly, as the precise epidemic period cannot be resolved due to the large confidence intervals of our estimates (>50 years). 

The next period in which we identified _Y. pestis_ was in the latter half of the 14<sup>th</sup> century. A third isolate from Ribe was dated to 1370 (1336-1408) and strongly clustered (posterior: 0.99) with post-Black Death samples from The Netherlands and Russia. These samples have previously been attributed to the _pestis secunda_ (1357-1366) [@namouchi2018IntegrativeApproachUsing], although we find the _pestis tertia_ (1364-76) [@slavin2021OutWestFormation] to be an equally likely candidate. This clade also has broader epidemiological significance, as it is directly ancestral to the Third Pandemic of plague (19<sup>th</sup>-20<sup>th</sup> century) [@spyrou2019PhylogeographySecondPlague]. Our results therefore reveal new global connections, as the same lineage that afflicted medieval Danish populations would later re-emerge to cause modern epidemics of plague, including the recent outbreaks in Madagascar [@nguyen20182017PlagueOutbreak].

We observed a curious gap in the continuity of plague at Ribe, as no _Y. pestis_ was detected there between 1408 and 1484. This was surprising, as 86% of individuals (43/50) from this site were archaeologically dated to between 1400 and 1536. Instead, the distribution of _Y. pestis_ appeared to shift during this period from the eastern coast of Jutland to the western coast. We recovered 3 distinct, and possibly contemporaneous, isolates of _Y. pestis_ from 3 sites near Horsens dated to 1429 (1392-1467), 1433 (1403-1464) and 1457 (1427-1487). These genomes were most closely related to individuals sampled in Germany, Lithuania, Poland, and England. This geographic association parallels the historical record, in which outbreaks in Denmark coincided with those in the Baltic region which often appeared first in Germany [eckert1978BoundaryFormationDiffusion; @slavin2021OutWestFormation]. However, recent studies have demonstrated that the directionality and spread of zoonotic diseases cannot be inferred from genomic data alone [@kalkauskas2021SamplingBiasModel; @eaton2021PlaguedCrypticClock]. Instead, our results establish an epidemiological link between _Y. pestis_ and historical case records in Denmark, which can be jointly modeled  [@duchene2019InferringInfectiousDisease] with greater nuance in future work.

In the 16<sup>th</sup> century, we once again observed _Y. pestis_ at Ribe GrÃ¤brÃ¸dre. Individuals G.207 
and G861 x1035 were dated to 1513 (95% HPD: 1484-1546) and 1525 (95% HPD: 1494-1560) respectively. Furthermore, we also found evidence of _Y. pestis_ in the northern site of Faldborg dated to 1594 (95% HPD: 1550-1649).

<div style="page-break-after: always;"></div>

![](https://rawcdn.githack.com/ktmeaton/plague-phylogeography-projects/802e051/denmark/auspice/all/chromosome/full/filter30/beast/timetree_map_timeline.png)

<div style="page-break-after: always;"></div>

## Materials and Methods {.page_break_before}

We sampled 298 individuals across 13 archaeological sites in Denmark (Figure 1A, Dataset S1). Site occupation dates spanned from the 11<sup>th</sup> to the 19<sup>th</sup> century CE. We estimated individual date ranges based on burial position, which was categorized according to cultural shifts that occurred in Denmark throughout the medieval and early modern period [@boldsen2009LeprosyMedievalDenmark]. When the original stratigraphic context was preserved, we refined these individual estimates further. For individuals with ambiguous or conflicting archaeological estimates, we performed radiocarbon dating when additional destructive sampling was permitted.

DNA was extracted from teeth and dental pulp according to a specialized protocol for ancient DNA [@dabney2013CompleteMitochondrialGenome]. Reagent blanks were introduced as negative controls to monitor DNA contamination in subsequent steps. We screened for plague using a PCR assay that targets the _pla_ virulence gene in _Yersinia pestis_ [@wagner2014YersiniaPestisPlague]. Extracts showing amplification in at least 4/6 replicates were converted into paired-end sequencing libraries [@kircher2012DoubleIndexingOvercomes]. Targeted capture of the _Y. pestis_ genome was performed using previously designed probes [@wagner2014YersiniaPestisPlague] and sequenced on an Illumina platform.

Sequenced molecules were aligned to a reference genome using the _[[nf-core/eager]]_ pipeline [@yates2021ReproduciblePortableEfficient]. To phylogenetically place these new samples, we downloaded a comparative dataset of 40 high-coverage _Y. pestis_ genomes (>3X) dated to the Second Pandemic. We then constructed a multiple alignment with the [[snippy]] pipeline [@snippy2020], which included 356 variation positions and 4,289,810 constant sites.

To tip-date each genome, we performed a [[Bayesian Evaluation of Temporal Signal]] (BETS [@duchene2020BayesianEvaluationTemporal], assuming a constant population size and using either a strict clock or an uncorrelated lognormal (UCLN) relaxed clock with _[[BEAST2]]_ [@bouckaert2019BEASTAdvancedSoftware]. Diffuse normal priors were constructed for all tip-dates, using the mean radiocarbon/mortuary date and half the uncertainty as the standard deviation. All Danish samples were assigned equivalent priors with a mean date of 1330 CE and a standard deviation of 115 years. Bayes factors were calculated by comparing the marginal likelihoods of each candidate model, as estimated with a generalized stepping stone (GSS) computation. The model with the highest marginal likelihood was then run for 100,000,000 generations to ensure the effective sample size (ESS) of all relevant parameters was greater than 200. 

Data visualization was performed using the python package _seaborn_ and _auspice_, a component of the Nextstrain visualization suite [@hadfield2018NextstrainRealtimeTracking].

## Data Availability

Raw sequence reads have have been deposited in NCBI BioProject PRJNAXXXXX. Archaeological metadata is provided in the supplementary information (Dataset SI).


## Acknowledgments

To Be Done :)

## References {.page_break_before}

<div id="refs"></div>

## SI {.page_break_before}

These are additional figures/data I anticipate co-authors or reviewers may want in the SI:
- Date distributions by site (1 page, 13 subplots)
- Maximum-likelihood phylogeny (1 page)
- Individual priors (1 page, ~20 subplots)