
---
title: 'Phylogeography Of The Second Plague Pandemic Revealed Through Analysis Of Historical Yersinia Pestis Genomes'
author: Spyrou
year: 2019
aliases: 
  - Spyrou et al. 2019
  - Spyrou et al. (2019)
  - '@spyrou2019PhylogeographySecondPlague'
---

# Spyrou 2019 Phylogeography Of The Second Plague Pandemic Revealed Through Analysis Of Historical *[[Yersinia Pestis]]* Genomes

## Seldon Summary

> - We used the Bayesian framework BEAST v1.8 in order to make an assessment of substitution rate variations across the genealogy of Branch 1 (n = 80), retaining high-quality second pandemic Y. pestis genomes and using available calibration points in our modern and ancient datasets (Supplementary Data 6).
> - Here, using previously published8–10,14 and new whole-genome data from 20 archaeological sites, we identify that all genomes associated with postBD outbreaks in Europe derived from a single ancestral strain that was present in southern, central, western and northern Europe during the BD.
> - For the visualisation of an identified deletion across BED and OBS isolates, we computed the average coverage across 3,000-bp windows in representative Y. pestis genomes from all analysed periods of the second pandemic, and subsequently used the program Circos90 to produce coverage plots of a 20-fold maximum coverage.
> - To investigate the genomic profiles of all newly reconstructed genomes, we analysed the presence or absence of potential virulence-associated and evolutionary determinant genes located on the Y. pestis chromosome (Fig. 4a) and plasmids (Supplementary Fig. 20)36,37, in comparison to published representatives of ancient and modern strains.
> - In order to calculate the substitution rate variation across Y. pestis isolates associated with the second pandemic, we first assessed the temporal signal across Branch 1 that includes all genomes from both the second and third plague pandemics.
> - As such, in order to avoid erroneous SNP assignments, we qualitatively evaluated all private SNP calls for the newly reconstructed genomes that were used for phylogenetic analysis in this study (minimum 50% of the genome covered 5-fold (Table 1)).
> - We find a number of genomes grouping with the previously described “post-BD” lineage together with published strains from Ellwangen (ELW098/549_O), Germany (1486–1630)9, and Marseille, France (1720–1722)14, which are descended from the European BD isolates (Fig. 2; Supplementary Data 1).