---
aliases:
  - Zhgenti et al. 2015
  - Zhgenti et al. (2015)
  - '@zhgenti2015GenomeAssemblies11'
---

# Zhgenti 2015 Genome Assemblies 11

| Field   | Value                                                                                                       |
| ------- | ----------------------------------------------------------------------------------------------------------- |
| Title   | [[Genome]] [[Assembly\|assemblies]] for 11 [[Yersinia pestis]] strains isolated in the [[Caucasus\|Caucasus Region]]. |
| Year    | 2015                                                                                                        |
| Authors | Zhgenti et al.                                                                                              |
| DOI     | <https://doi.org/10.1128/genomeA.01030-15>                                                                                          |
| bibtex  | zhgenti2015GenomeAssemblies11                                                                               |


## Summary

- A [[Genome|genome]] announcement for the sequencing of 11 [[Yersinia pestis|Y. pestis]] strains from the [[Caucasus]], specifically:
	-  [[Georgia]], [[Armenia]], [[Azerbaijan]], [[Kyrgyzstan]].
-  These regions are [[Under-represented]].
-  The [[Caucasus]] is a historically important region because it sits at the crossroads of the [[Asia]] and [[Europe]] along the [[Silk Road|Great Silk Road]]. As a result, this region has been visited by many [[Epidemic|epidemics]], including [[Plague|plague]], with the earliest mention in the [[11th-Century]].
- The authors credit the surveillance study work of [[Bakanidze et al. 2010 Surveillance Plague Natural|Bakanidze et al. (2010)]] for the [[11th-Century]] mention of [[Plague|plague]].
-  The [[Caucasus]] has very unique [[Clade|strains]] of [[Plague|plague]], referencing the work of [[Anisimov et al. 2004 Intraspecific Diversity Yersinia|Anisimov et al. (2004)]], [[Gage and Kosoy 2005 Natural History Plague|Gage and Kosoy (2005)]], and [[Revazishvili et al. 2008 Characterisation Yersinia Pestis|Revazishvili et al. (2008)]].
- This is an important contributions to reducing [[Sampling Bias]].
- These [[Genome|genomes]] will be included by [[Spyrou 2016 Historical Pestis Genomes|Spyrou et al. (2016)]] in their study.

## Tables

|                                                        Table 1                                                        |     |
|:---------------------------------------------------------------------------------------------------------------------:| --- |
|                                     ![[zhgenti2015GenomeAssemblies11_table1.jpg]]                                     |     |
| [[Genomic\|genomic]] metadata for the [[Yersinia pestis\|Y. pestis]] strains [[Sequencing\|sequenced]] in this study. |     |
[[Zhgenti 2015 Genome Assemblies 11\|Zhgenti et al. (2015)]] | 	|	

## Notes

---

tags: #ðŸ“¥/ðŸ“°

## Seldon Summary

> - The strains sequenced in this effort were selected from a historic collection of Y. pestis isolates from the Caucasus region preserved in the repository of the National Center for Disease Control and Public Health of Georgia.
> - Here, we present the improved draft or finished assembled genomes from 11 strains isolated in the nation of Georgia and surrounding countries.
> - Genome assemblies for 11 Yersinia pestis strains isolated in the Caucasus region.
> - Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data.
> - Characterisation of Yersinia pestis isolates from natural foci of plague in the Republic of Georgia, and their relationship to Y. pestis isolates from other countries.
> - The raw Illumina reads were assembled in Velvet, and the Velvet consensus sequences were computationally shredded into 1.5-Kbp overlapping shreds (8).
> - High-quality genomic DNA was extracted from purified isolates of each strain and draft sequence data generated using a combination of Illumina and 454 technologies (5, 6).