---
aliases:
  - Zimpel et al 2020
  - Zimpel et al. (2020)
  - '@zimpel2020GlobalDistributionEvolution'
type: [[Literature]]
tags:
  - üìù/üåø
  - üì•/üì∞ 
---

# Zimpel 2020 Global Distribution Evolution

| Field   | Value                                                               |
| ------- | ------------------------------------------------------------------- |
| Title   | Global Distribution and Evolution of *Mycobacterium bovis* Lineages |
| Year    | 2020                                                                |
| Authors | Zimpel et al. 2020                                                  |
| DOI     | <https://doi.org/10.3389/fmicb.2020.00843>                          | 
| bibtex  | zimpel2020GlobalDistributionEvolutiona                              |


## Concepts

- [[Phylogenetic]]
- [[Genome]]
- [[Mycobacterium]] [[Mycobacterium bovis|bovis]]
- [[Clade]]
- [[Global]]

## Summary

- Identify at least four [[Global]] lineages of [[Mycobacterium bovis]].
- [[Clade|Lineages]] cluster by location rather than host species, indicating a generalist member of the [[Mycobacterium Tuberculosis Complex|MTBC]].
- [[Clade|Lineaege]] have independent evolutionary trajectories that underly the current distribution of [[Tuberculosis]].
- Sampling biases such as the lack of genomes from [[Africa]], [[Asia]], and [[Continental Europe]] prevent precise estimation of geographic origin.

There are many similarities between this and [[Plague Phylodynamics and Phylogeography]], and thus is a useful reference for formatting and content.

## Notes

1. Introduction
	  - Introduce [[Tuberculosis]] the disease and it's impact.
	  - Introduce the pathogen, [[Mycobacterium bovis]], ecotypes, subspecies, ecology, distribution. Evolutionary history and mechanisms.
	  - Introduce the topic, subtyping and [[Clade|lineages]] and what is known so far.
	  - Introduce the problem:
		> *...no studies to date have comprehensively analyzed _M. bovis_ genomes at a global scale to provide insights into its populational structure and evolution based on whole-genome information.*
		
1. Materials and Methods
	- They do genome sequencing, and so it is described.
	- Data Collection: Search keywords and selection criteria.
	- Sequencing Quality Criteria/Mapping/Variant Calling
	- Phylogenetic Reconstruction
	- Subsampling. To have a more even representation of genomes and to reduce redundancy, [[Mycobacterium bovis]] genomes were subsampled using [[Treemer]].
	- Phylogenetic clustering and [[Principal Component Analysis]].
	- [[Dating]] divergences.

1. Results
	- [[Host|Hosts]] are dispersed throughout the tree in different clades. The majority of genomes show geographic clustering at the [[Continent]] level.
	
	> *This finding suggests that geographic proximity between wildlife and bovine hosts and their contact rates has played a more important role in determining host range of _M. bovis_ than phylogenetic distance among hosts*

	> The lack of host clustering supports the hypothesis of _M. bovis_ being a generalist member of MTBC, able to infect different host species irrespective of the bacterial genetic makeup.

	- Four countries were over-represented, with samples forming clusters of highly similar genomes. Thus it was helpful to use [[Treemer]] to remove redundancy in the dataset.
	
	>After removal, we carefully checked to guarantee that all countries were still represented and that all _M. bovis_ genomes from small clades were still part of the dataset.
	- This is how I subsampled [[Yersinia pestis|Y. pestis]] for [[Plague Phylodynamics and Phylogeography]], ensuring all locations and clades were retained.
	- They describe the [[Clade|clades]] in the tree, and their natural history of emergence and [[Distribution]].
	- They compare [[Clock Model|clock models]] with regards to [[Node Dating]] and [[Substitution Rate]]. And identify which publications this agrees with [[Bos et al. 2014]].

1. Conclusion

## Figures

|          Fig 3          |
|:-----------------------:|
| ![\|600](https://www.frontiersin.org/files/Articles/509125/fmicb-11-00843-HTML/image_m/fmicb-11-00843-g003.jpg) |
|     Phylogenetic clustering of *[[Mycobacterium bovis]]*.     |
[[Zimpel 2020 Global Distribution Evolution\|Zimpel et al. (2020)]] |

|          Fig 5          |
|:-----------------------:|
| ![\|600](https://www.frontiersin.org/files/Articles/509125/fmicb-11-00843-HTML/image_m/fmicb-11-00843-g005.jpg) |
|     Dating estimates and molecular rates of evolution of *[[Mycobacterium bovis]]*.     |
[[Zimpel 2020 Global Distribution Evolution\|Zimpel et al. (2020)]] |