<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <title>Chromosome Resequencing</title>
  <style type="text/css">
      code{white-space: pre-wrap;}
      span.smallcaps{font-variant: small-caps;}
      span.underline{text-decoration: underline;}
      div.column{display: inline-block; vertical-align: top; width: 50%;}
  </style>
  <style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
  </style>
</head>
<body>
<h1 id="chromosome-resequencing">Chromosome Resequencing</h1>
<table>
<thead>
<tr class="header">
<th>Field</th>
<th>Value</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Project</td>
<td>[[Plague Denmark]]</td>
</tr>
<tr class="even">
<td>Date</td>
<td>2021-APR</td>
</tr>
</tbody>
</table>
<hr />
<h2 id="objectives">Objectives</h2>
<ol type="1">
<li>Identify [[Yersinia pestis]]-positive samples for [[Resequencing]] based on:
<ul>
<li>[<a href="#library-complexity">Library complexity</a>]</li>
<li>[[Genome coverage]], specifically the [[Chromosome]].</li>
<li>Informative [[SNP|SNPs]].</li>
</ul></li>
</ol>
<hr />
<h2 id="conclusions">Conclusions</h2>
<ol type="1">
<li>The following enriched libraries are good candidates for resequencing:
<ul>
<li>D24, D62, D72, R21</li>
</ul></li>
<li>The threshold of 50% coverage at 3X is a relatively good predictor of reaching the minimum number of informative [[SNP|SNPs]] for [[Phylogeny|phylogenetic]] applications.</li>
<li>All of the libraries that [<span class="citation" data-cites="Ravneet">[@Ravneet Sidhu|Ravneet]</span>] has enriched have drastically better complexity than libraries enriched by [<span class="citation" data-cites="Katherine">[@Katherine Eaton|Katherine]</span>]. Some factors could be:
<ul>
<li>Different samples, with different preservation and infectious load.</li>
<li>Different baitsets, [<span class="citation" data-cites="Ravneet">[@Ravneet Sidhu|Ravneet]</span>] was the first to use the newly synthesized core baits.</li>
<li>Different technique, [<span class="citation" data-cites="Ravneet">[@Ravneet Sidhu|Ravneet]</span>] may have better laboratory technique somewhere in the workflow :)</li>
</ul></li>
</ol>
<hr />
<h2 id="overview">Overview</h2>
<ol type="1">
<li>Run the [[plague-phylogeography]] [[snakemake]] pipeline for [[Medieval]] [[Denmark]] samples.</li>
<li>Calculate library complexity: [[Preseq]].</li>
<li>Calculate the number of informative SNPs: [[Snippy]].</li>
<li>Calculate genome coverage: [[Qualimap]].</li>
<li>Create charts: [[Plotly]].</li>
<li>Identify samples for [[Resequencing]].</li>
</ol>
<pre class="mermaid"><code>graph TD;
1--&gt;2;
2--&gt;3;
2--&gt;4;
4--&gt;6;
2--&gt;5;
5--&gt;6;
3--&gt;6;

1[Load Project]
2[Snakemake Pipeline]
3[Library Complexity]
4[Genome Coverage]
5[Informative SNPs]
6[Plotly]
</code></pre>
<hr />
<h2 id="results">Results</h2>
<h3 id="genome-coverage-and-snps">Genome Coverage and SNPs</h3>
<p>This stats table reflects the [[Genome coverage]] and [[SNP]]s merged across all libraries for a sample. The [[Black Death]] sample [[SAMN00715800|8291]] is included for context as a high coverage genome, and setting the minimum expected number of [[SNP|SNPs]].</p>
<p><code>Coverage Status</code> is set to 0 if the sample is below 50% and will be colored red on the following chart.</p>
<p><code>SNP Status</code> is set to 0 if the sample is below 90 SNPs and will be colored red on the following chart.</p>
<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://chart-studio.plotly.com/~ktmeaton/0.embed" height="525" width="100%">
</iframe>
<p><br> <iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://chart-studio.plotly.com/~ktmeaton/1.embed" height="525" width="100%"></iframe></p>
<ul>
<li>The [[Coverage]] of the reference and the Number of [[SNP|SNPs]] is highly related.</li>
<li>The threshold of 50% coverage at 3X is a relatively good predictor of reaching the minimum number of SNPs (relative to [[Black Death]] [[SAMN00715800|8291]]).</li>
<li>Three samples have high coverage (70%+) and do not need resequencing.
<ul>
<li>D51, D71, D75, R36</li>
</ul></li>
<li>Five samples have moderate coverage (50-70%) which could be improved by resequencing.
<ul>
<li>D62, D72, P187, P212, P387</li>
</ul></li>
<li>Three samples have low coverage and moderate SNPs. Depending on the [[complexity curves]], they may be good candidates for resequencing.
<ul>
<li>D24, P384, R21</li>
</ul></li>
<li>Two samples have extremely low coverage and few SNPs. These are NOT good candidates for resequencing.
<ul>
<li>D25, P246, R44</li>
</ul></li>
</ul>
<hr />
<h3 id="library-complexity">Library Complexity</h3>
<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://chart-studio.plotly.com/~ktmeaton/33.embed" height="525" width="100%">
</iframe>
<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://chart-studio.plotly.com/~ktmeaton/35.embed" height="525" width="100%">
</iframe>
<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://chart-studio.plotly.com/~ktmeaton/39.embed" height="525" width="100%">
</iframe>
<p>Based on the [[Chromosome Resequencing#Genome Coverage and SNPs|Genome Coverage and SNPs]], these plots are being used to evaluate whether medium and low coverage samples should be resequenced.</p>
<table>
<thead>
<tr class="header">
<th>Category</th>
<th>Sample</th>
<th>Description</th>
<th>Action</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Medium</td>
<td>D62</td>
<td>Still has an upward trajectory.</td>
<td>Resequence</td>
</tr>
<tr class="even">
<td>Medium</td>
<td>D72</td>
<td>Starting to plateau.</td>
<td>Resequence</td>
</tr>
<tr class="odd">
<td>Medium</td>
<td>P187</td>
<td>Plateaued with few unique molecules.</td>
<td>None</td>
</tr>
<tr class="even">
<td>Medium</td>
<td>P212</td>
<td>Plateaued with few unique molecules.</td>
<td>None</td>
</tr>
<tr class="odd">
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Low</td>
<td>P384</td>
<td>Plateaued with few unique molecules.</td>
<td>None</td>
</tr>
<tr class="odd">
<td>Low</td>
<td>P387</td>
<td>Plateaued with few unique molecules.</td>
<td>None</td>
</tr>
<tr class="even">
<td>Low</td>
<td>R21</td>
<td>Similar to D72.</td>
<td>Resequence</td>
</tr>
<tr class="odd">
<td>Low</td>
<td>D24</td>
<td>Marginally worse than D72.</td>
<td>Resequence</td>
</tr>
</tbody>
</table>
<hr />
<h2 id="methods">Methods</h2>
<h3 id="data-preparation">Data Preparation</h3>
<ol type="1">
<li><p>Clone the [[plague-phylogeography]] projects repository.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb2-1" title="1"><span class="fu">git</span> clone https://github.com/ktmeaton/plague-phylogeography-projects.git /2/scratch/keaton/plague-phylogeography-projects</a></code></pre></div></li>
<li><p>Load the [[Plague Denmark]] into the [[plague-phylogeography]] [[snakemake]] pipeline.</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb3-1" title="1"><span class="ex">workflow/scripts/project_load.sh</span> results ../plague-phylogeography-projects/denmark rsync</a></code></pre></div></li>
</ol>
<h3 id="analysis">Analysis</h3>
<ol type="1">
<li><p>Run <code>eager local</code> to generate [[Preseq]] output.</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb4-1" title="1"><span class="ex">snakemake</span> eager_local --profile profiles/infoserv --configfile results/config/snakemake.yaml</a></code></pre></div></li>
<li><p>Create a [[MultiQC]] report.</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb5-1" title="1"><span class="ex">snakemake</span> multiqc_local --profile profiles/infoserv --configfile results/config/snakemake.yaml</a></code></pre></div>
<p>*Note: This output was very messy and complicated. I used [[Plotly]] instead to make individual charts.</p></li>
<li><p>Prep tables for plotly.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb6-1" title="1"><span class="bu">cd</span> plague-phylogeography-projects/denmark/multiqc/local/multiqc_data</a>
<a class="sourceLine" id="cb6-2" title="2"><span class="fu">mkdir</span> -p plotly</a>
<a class="sourceLine" id="cb6-3" title="3"><span class="bu">cd</span> plotly</a>
<a class="sourceLine" id="cb6-4" title="4"><span class="ex">../../../../../scripts/plotly_preseq.sh</span> ../mqc_preseq_plot_1.txt</a></code></pre></div>
<p>The following samples had multiple libraries:</p>
<ul>
<li>P187</li>
<li>P212</li>
<li>P246</li>
<li>P384</li>
<li>P387</li>
</ul>
<p>There appeared to be little significant variation between the libraries. I chose the M4 experiment on library a to be representative of each.</p>
<p>*Note: I just made the “Perfect” one manually.</p></li>
<li><p>Create a [[snakemake]] report.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb7-1" title="1"><span class="fu">mkdir</span> -p results/report/local/</a>
<a class="sourceLine" id="cb7-2" title="2"><span class="ex">snakemake</span> multiqc_local --report results/report/local/report.html --profile profiles/infoserv --configfile results/config/snakemake.yaml</a></code></pre></div></li>
</ol>
<h3 id="upload-results">Upload Results</h3>
<ol type="1">
<li><p>Unload project results.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb8-1" title="1"><span class="ex">workflow/scripts/project_unload.sh</span> results ../plague-phylogeography-projects/denmark rsync</a></code></pre></div></li>
<li><p>Upload project results.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode bash"><code class="sourceCode bash"><a class="sourceLine" id="cb9-1" title="1"><span class="bu">cd</span> ../plague-phylogeography-projects/</a>
<a class="sourceLine" id="cb9-2" title="2"><span class="fu">git</span> add -A</a>
<a class="sourceLine" id="cb9-3" title="3"><span class="fu">git</span> commit -m <span class="st">&quot;&quot;</span></a>
<a class="sourceLine" id="cb9-4" title="4"><span class="fu">git</span> push origin</a></code></pre></div></li>
</ol>
<hr />
<p>tags: [[Experiment]]</p>
</body>
</html>
